<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2021-09-11 Sat 09:34 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Human Diseases</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Houjun Liu" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: auto;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline; margin-top: 14px;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" href="https://taproot3.sanity.gq/global.css" type="text/css"/>
<script type="text/javascript">
// @license magnet:?xt=urn:btih:e95b018ef3580986a04669f1b5879592219e2a7a&dn=public-domain.txt Public Domain
<!--/*--><![CDATA[/*><!--*/
     function CodeHighlightOn(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.add("code-highlighted");
         target.classList.add("code-highlighted");
       }
     }
     function CodeHighlightOff(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.remove("code-highlighted");
         target.classList.remove("code-highlighted");
       }
     }
    /*]]>*///-->
// @license-end
</script>
<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        displayAlign: "center",
        displayIndent: "0em",

        "HTML-CSS": { scale: 100,
                        linebreaks: { automatic: "false" },
                        webFont: "TeX"
                       },
        SVG: {scale: 100,
              linebreaks: { automatic: "false" },
              font: "TeX"},
        NativeMML: {scale: 100},
        TeX: { equationNumbers: {autoNumber: "AMS"},
               MultLineWidth: "85%",
               TagSide: "right",
               TagIndent: ".8em"
             }
});
</script>
<script type="text/javascript"
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_HTML"></script>
</head>
<body>
<div id="org-div-home-and-up">
 <a accesskey="h" href="https://taproot3.sanity.gq/sitemap.html"> UP </a>
 |
 <a accesskey="H" href="https://taproot3.sanity.gq/theindex.html"> HOME </a>
</div><div id="content">
<h1 class="title">Human Diseases</h1>


<div id="outline-container-overview-of-human-diseases" class="outline-2">
<h2 id="overview-of-human-diseases"><span class="section-number-2">1</span> Overview of Human Diseases</h2>
<div class="outline-text-2" id="text-overview-of-human-diseases">
<p>
A lecture by the Legendary Dr.Â Paul Hauser.
<a href="https://docs.google.com/presentation/d/1b2RetU6iGsd_h4Msb2SV-_WznNXSREbsPpfdY-LgJZs/edit#slide=id.ga6d683dbbf_0_338">Slides
are here</a>
</p>

<p>
#flo #disorganized
</p>

<p>
<b>Disease</b> is an abnormal condition that causes impairment in/loss of
function of an organism (a.k.a. decreased fitness) that is not due to
immediate external injury.
</p>

<ul class="org-ul">
<li>What causes human disease?

<ul class="org-ul">
<li>Infectious agents</li>
<li>Deficiency disorders</li>
<li>Heritable factors</li>
<li>Physiological disorders (immunodeficiency, autoimmune disorders,
allergies, etc.)</li>
</ul></li>
</ul>
</div>

<div id="outline-container-congenital-vs.-acquired-disease" class="outline-3">
<h3 id="congenital-vs.-acquired-disease"><span class="section-number-3">1.1</span> Congenital vs.Â Acquired disease</h3>
<div class="outline-text-3" id="text-congenital-vs.-acquired-disease">
<p>
Congenital diseases =&gt; diseases present at birth due to DNA
abnormalities / pregnancy pathological issues
</p>

<p>
Acquired diseases =&gt; diseases that begin during lifetime, including&#x2026;
</p>

<ul class="org-ul">
<li>Microrganism invasion =&gt; "infectios diseases"</li>
<li>Autoimmune reaction</li>
<li>Nutrient deficiency</li>
<li>Mechanical wear</li>
<li>Ingestion of noxious chemicals</li>
</ul>

<p>
<b>Infectious diseases actually smaller on the causes of death in the US</b>
</p>

<ul class="org-ul">
<li>Heart disease =&gt; wear + deficiency</li>
<li>Cancer =&gt; heritable + DNA</li>
<li>Unintentional injuries =&gt; not a disease</li>
<li>Chronic respitory disease =&gt; wear</li>
<li>Stroke =&gt; not a disease</li>
<li>Alhetimer disease =&gt; wear</li>
<li>Diabetes =&gt; autoimmune, nutrient, wear</li>
<li>Influenca &lt;= <b>here, finally, an infections disease.</b></li>
</ul>
</div>
</div>

<div id="outline-container-disease-causing-agents" class="outline-3">
<h3 id="disease-causing-agents"><span class="section-number-3">1.2</span> Disease causing agents</h3>
<div class="outline-text-3" id="text-disease-causing-agents">
<ul class="org-ul">
<li><b>Protozoan</b> =&gt; single-celled eukaryotes</li>
<li><b>Fungal</b> =&gt; single/multi-celled eukarotyes</li>
<li><b>Bacteria</b> =&gt; single-celled prokaryotes</li>
<li><b>Viral</b> =&gt; acellular parasitic infectious agent</li>
<li><b>Helminuthus</b> =&gt; multicellular worms</li>
<li><b>Prions</b> =&gt; acellular misfolded proteins</li>
<li><b>Viroids</b> =&gt; infections nucleic acids w/o protein coat to make virus</li>
</ul>
</div>
</div>

<div id="outline-container-pathogenicity-virulence" class="outline-3">
<h3 id="pathogenicity-virulence"><span class="section-number-3">1.3</span> Pathogenicity + Virulence</h3>
<div class="outline-text-3" id="text-pathogenicity-virulence">
<p>
<b>Pathoginecity</b> =&gt; relative capacity to cause disease
</p>

<ul class="org-ul">
<li>Non-pathogenic agents =&gt; no diesease</li>
<li>Primary pathogens =&gt; yes disease</li>
<li>Opportunistic pathogens =&gt; yes disease only when it can, for instance,
in immunocompromised individuals</li>
</ul>

<p>
<b>Virulence</b> =&gt; numerical measures for pathonicity
</p>

<ul class="org-ul">
<li>Measured experimentally with LD50 + ED50</li>
</ul>

<hr />
</div>
</div>

<div id="outline-container-overview-of-various-diseases" class="outline-3">
<h3 id="overview-of-various-diseases"><span class="section-number-3">1.4</span> Overview of various diseases</h3>
<div class="outline-text-3" id="text-overview-of-various-diseases">
<p>
<a href="https://drive.google.com/file/d/1WRnbgkhnmRrdP4ZqlqT3HHBD1_eW9qib/view">This
video</a>
</p>
</div>

<div id="outline-container-protozoan" class="outline-4">
<h4 id="protozoan"><span class="section-number-4">1.4.1</span> Protozoan</h4>
<div class="outline-text-4" id="text-protozoan">
<ul class="org-ul">
<li><b>Protozoan factors</b> =&gt; direction pathogenisis leading to tissue damage</li>
<li><b>Host-mediated factors</b> =&gt; immune evation + escape mechnisms +
immunalsupression</li>
</ul>

<p>
Adaptable!!
</p>
</div>
</div>

<div id="outline-container-fungal" class="outline-4">
<h4 id="fungal"><span class="section-number-4">1.4.2</span> Fungal</h4>
<div class="outline-text-4" id="text-fungal">
<ul class="org-ul">
<li><b>Fungal factors</b> =&gt; many shapes and very adaptable, colud produced
specialized enzymes to take root in body</li>
<li><b>Host-mediated factors</b> =&gt; cause immunocomprimzation, acquired though
inhalation, etc.</li>
</ul>
</div>
</div>

<div id="outline-container-bacteria" class="outline-4">
<h4 id="bacteria"><span class="section-number-4">1.4.3</span> Bacteria</h4>
<div class="outline-text-4" id="text-bacteria">
<ul class="org-ul">
<li><b>Bacterial-induced toxicity</b> =&gt; produces toxins + has hard capsule
cell</li>
<li><b>Host-mediated factors</b> =&gt; may develop host resistance, could compete
for resources, and could be grown introcellularly</li>
</ul>


<div id="org611f7ae" class="figure">
<p><img src="Screen Shot 2020-10-12 at 3.08.53 PM.png" alt="Screen Shot 2020-10-12 at 3.08.53 PM.png" />
</p>
<p><span class="figure-number">Figure 1: </span>Screen Shot 2020-10-12 at 3.08.53 PM.png</p>
</div>

<hr />
</div>
</div>
</div>

<div id="outline-container-bacteria-causing-diseases" class="outline-3">
<h3 id="bacteria-causing-diseases"><span class="section-number-3">1.5</span> Bacteria causing diseases</h3>
<div class="outline-text-3" id="text-bacteria-causing-diseases">
<p>
<b>Biofilm formation</b>
</p>

<ul class="org-ul">
<li>Communities of bacteria could work together by adhering and exchanging
information</li>
<li>Bacterial could perform quorum sensing =&gt; exchange of information with
each other + recognize various members of their group</li>
</ul>
</div>

<div id="outline-container-fighting-bacterial-infections" class="outline-4">
<h4 id="fighting-bacterial-infections"><span class="section-number-4">1.5.1</span> Fighting bacterial infections</h4>
<div class="outline-text-4" id="text-fighting-bacterial-infections">
<p>
<b>Antibiotics</b> =&gt; drugs with selective toxicity for specific bacterial
types
</p>

<p>
Act by&#x2026;
</p>

<ul class="org-ul">
<li>Disrupting membrane + cell wall integrity</li>
<li>Selectively target + impair bacterial ribosomes</li>
<li>Block bacterial DNA replication/transcription</li>
<li>Inhibit bacterial metabolism</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-viruses-causing-diseases" class="outline-3">
<h3 id="viruses-causing-diseases"><span class="section-number-3">1.6</span> Viruses causing diseases</h3>
<div class="outline-text-3" id="text-viruses-causing-diseases">
<p>
<b>Viruses: acellular macromolecular assemblies</b>
</p>

<ul class="org-ul">
<li>Contain protein coat called capsid</li>

<li>DNA or RNA, but not both</li>

<li>Are obligate parasites that could only replicate within host</li>

<li>Assembled and mature viral particles =&gt; virions, which contain&#x2026;

<ul class="org-ul">
<li>Capsid</li>
<li>Genetic material</li>
<li>Occationally outside lipid layer</li>
</ul></li>
</ul>

<p>
=&gt; Viruses exist on the nanometre scale, but they are difference in
share and size
</p>
</div>

<div id="outline-container-structure-of-viruses" class="outline-4">
<h4 id="structure-of-viruses"><span class="section-number-4">1.6.1</span> Structure of viruses</h4>
<div class="outline-text-4" id="text-structure-of-viruses">
<ul class="org-ul">
<li><b>All contain</b>

<ul class="org-ul">
<li>Capsid =&gt; structural protein coat</li>
<li>Genome =&gt; RNA/DNA; but not both</li>
</ul></li>

<li><b>Some contain</b>

<ul class="org-ul">
<li>Membraneous-enclosed capsid =&gt; envelope</li>
<li>Externally-facisg host-cell fusion proteins =&gt; spikes</li>
<li>Viral genome replication enzymes =&gt; prlymerases</li>
<li>Other proteins for fun =&gt; enzymes, motor proteins, transcription
factors, host-cell interacting proteins, etc.</li>
</ul></li>
</ul>
</div>
</div>

<div id="outline-container-two-types-of-virus" class="outline-4">
<h4 id="two-types-of-virus"><span class="section-number-4">1.6.2</span> Two types of virus</h4>
<div class="outline-text-4" id="text-two-types-of-virus">
<ul class="org-ul">
<li><b>Prokaryotic-infecting viruses</b>

<ul class="org-ul">
<li>Variety of shapes</li>
<li>Complex and prolate shapes</li>
<li>Has, sometimes complex shapes! a la
<a href="Screen Shot 2020-10-12 at 10.49.04 PM.png">this
image</a></li>
</ul></li>

<li><b>Eukarotic-infecting viruses</b>

<ul class="org-ul">
<li>Much more "boring" in terms of shape</li>
<li>Icosahedral/spherecial outside</li>
<li>Enveloped constructions =&gt; envelope protein layer outside, spherical
inside</li>
<li>Helical/Cylindrical/Bullet shapes, too!</li>
<li>Often single patterns assemble together to create symmetric shape
that creates the whole of the virus</li>
</ul></li>
</ul>
</div>
</div>

<div id="outline-container-viral-life-cycle" class="outline-4">
<h4 id="viral-life-cycle"><span class="section-number-4">1.6.3</span> Viral Life Cycle</h4>
<div class="outline-text-4" id="text-viral-life-cycle">
<ol class="org-ol">
<li>Attachment =&gt; protein contact between virus and host</li>
<li>Viral entry/Uncoating =&gt; shedding the protein layer</li>
<li>Biosynthesis =&gt; make baby viruses

<ol class="org-ol">
<li>Genome Replication: transcribe DNA/RNA</li>
<li>Genome Expression: read DNA/RNA to make proteins</li>
</ol></li>

<li>Viral genome integration =&gt; retrovirus only</li>
<li>Assembly =&gt; put it all togethr</li>
<li>Viral Exit =&gt; mature virons leave</li>
</ol>
</div>

<ol class="org-ol">
<li><a id="viral-entry"></a>Viral Entry<br />
<div class="outline-text-5" id="text-viral-entry">
<p>
<i>Option 1: Direct Injection/insertion</i>
</p>

<ul class="org-ul">
<li>Insert genome through the bi-layer</li>
<li>Leave the rest behind</li>
<li>Tada!</li>
</ul>

<p>
<i>Option 2: Endocytosis</i>
</p>

<ul class="org-ul">
<li>Trick the host cell into introducing the virus as food</li>
<li>Endocytosis!</li>
<li>Bam</li>
</ul>

<p>
<i>Option 3: Fusion</i>
</p>

<ul class="org-ul">
<li>Virus fuse with cell membrane</li>
<li>Shed the protein coat once in</li>
<li>Shazam!</li>
</ul>

<p>
<b>All of these involve attachment first, which usually takes two steps.</b>
</p>

<p>
This process causes the organism-specific response to viruses:
</p>

<ol class="org-ol">
<li>Attachment: adhere roughly to random sugar proteins</li>
<li>Binding: roll over slowly, and bind to the entry receptor it needs</li>
</ol>
</div>
</li>

<li><a id="uncoating"></a>Uncoating<br />
<div class="outline-text-5" id="text-uncoating">
<ul class="org-ul">
<li>Virus triggers <i>early endosome</i>

<ul class="org-ul">
<li>Causes pH dependent protein denaturation</li>
<li>Causing the capsid to fall apart</li>
<li>Triggering <i>late endosome</i> =&gt; releasing genome</li>
</ul></li>
</ul>
</div>
</li>

<li><a id="viral-replication"></a>Viral Replication<br />
<div class="outline-text-5" id="text-viral-replication">
<p>
Key questions:
</p>

<ul class="org-ul">
<li><b>How are viral mRNAs produced from the viral genome?</b> =&gt; virus will
hijack the ribosomes in the host cells. So, it is more important to
ask how the mRNAs are produced to tell ribosomes what to do</li>
<li><b>What serves as the template for viral genome replication</b> =&gt;
replication will need a polymeraese; but the source and mechanism is
dependent on viral genome structure/composition</li>
</ul>


<div id="orgf8d29e2" class="figure">
<p><img src="Screen Shot 2020-10-12 at 11.04.53 PM.png" alt="Screen Shot 2020-10-12 at 11.04.53 PM.png" />
</p>
<p><span class="figure-number">Figure 2: </span>Screen Shot 2020-10-12 at 11.04.53 PM.png</p>
</div>

<p>
<b>DNA Viruses</b>
</p>

<p>
<i>How are viral mRNAs produced from the viral genome?</i>
</p>

<ul class="org-ul">
<li>Viral DNA enters, through RNA polymerase II in the host cell, mRNA is
produced</li>
<li>mRNAs then read by ribosomes, and there we go</li>
</ul>

<p>
<i>What serves as the templates for viral genome replication?</i>
</p>

<ul class="org-ul">
<li>Viral DNA serves as template for host cell DNA polymerase</li>
<li>Viral genome copied repeatedly</li>
<li>Virus, then, <b>will be replicated within the nucleus</b> due to it needing
the polymerase to copy DNA</li>
</ul>

<p>
Except! Poxvirade carry their own polymerase, so they replicate in the
cytoplasm.
</p>


<div id="orgd51b5b1" class="figure">
<p><img src="Screen Shot 2020-10-12 at 11.09.46 PM.png" alt="Screen Shot 2020-10-12 at 11.09.46 PM.png" />
</p>
<p><span class="figure-number">Figure 3: </span>Screen Shot 2020-10-12 at 11.09.46 PM.png</p>
</div>

<p>
<b>RNA Viruses</b>
</p>

<p>
<i>How are viral mRNAs produced from the viral genome?</i>
</p>

<ul class="org-ul">
<li>+Strand: reproducable RNA =&gt; could be directly translated by the
ribosomes</li>
<li>-Strand RNA: useless template RNA (less easy to be detected)

<ul class="org-ul">
<li>Need to be processed by RDRP (RNA-dependent RNA Polymerease)</li>
<li>Once entered the cell, RDRP goes to work copying -Strand RNA to
+Strang RNA</li>
</ul></li>

<li>double-stranded RNA viron =&gt; (+, a.k.a. sense)

<ul class="org-ul">
<li>+-stranded RNA =&gt; same idea as above</li>

<li><ul class="org-ul">
<li>strand RNA =&gt; virus comes with RDRP that convert -ssRNA to +ssRNA.
Then, same idea as above.</li>
</ul></li>
</ul></li>
</ul>

<p>
<i>What serves as the templates for viral genome replication?</i>
</p>

<ul class="org-ul">
<li>RNA viruses does not need host-cell polymeraese to copy RNA</li>
<li>They come with polymerase that&#x2026;

<ul class="org-ul">
<li>with dsRNA; takes +ssRNA and makes -ssRMA; combining the two to
produce dsRNA</li>
<li>with +ssRNA, takes +ssRNA and makes temporary -ssRNA which makes
more +ssRNA</li>
<li>with -ssRNA, takes -ssRNA, and makes temporary +ssRNA, which makes
-ssRNA</li>
</ul></li>
</ul>


<div id="org3813ff5" class="figure">
<p><img src="Screen Shot 2020-10-12 at 11.14.30 PM.png" alt="Screen Shot 2020-10-12 at 11.14.30 PM.png" />
</p>
<p><span class="figure-number">Figure 4: </span>Screen Shot 2020-10-12 at 11.14.30 PM.png</p>
</div>
</div>
</li>

<li><a id="packaging"></a>Packaging<br />
<div class="outline-text-5" id="text-packaging">
<p>
Does not require ATP. Just sealed in.
</p>
</div>
</li>

<li><a id="viral-exis"></a>Viral Exis<br />
<div class="outline-text-5" id="text-viral-exis">
<p>
<b>Lysis</b>
</p>

<p>
Replicate so much that the membrane burst.
</p>

<p>
<b>Budding</b>
</p>

<p>
Trigger&#x2026;
</p>

<ul class="org-ul">
<li>Trigger extocytosis</li>
<li>Meanwhile, send virus's own spikes to the membrane</li>
<li>On exit by extocytosis, steal a part of the newly-spikey membrane with
it to serve as new casing</li>
</ul>
</div>
</li>
</ol>
</div>

<div id="outline-container-viral-genetic-shift-viral-genetic-drift" class="outline-4">
<h4 id="viral-genetic-shift-viral-genetic-drift"><span class="section-number-4">1.6.4</span> Viral Genetic Shift + Viral Genetic Drift</h4>
<div class="outline-text-4" id="text-viral-genetic-shift-viral-genetic-drift">
<p>
<b>Shift</b> =&gt; whole segments of genome exchange abruptly as two flu viruses
infect the same cell to create a new strand.
</p>

<p>
<b>Drift</b> =&gt; single/groups of nucleotides flip slowly over time.
</p>

<p>
The former is an environment-dependent process, where the latter is able
to be modeled as it is due to transcription mistake.
</p>
</div>
</div>

<div id="outline-container-retroviruses-how-to-stop-them" class="outline-4">
<h4 id="retroviruses-how-to-stop-them"><span class="section-number-4">1.6.5</span> Retroviruses + How to Stop Them</h4>
<div class="outline-text-4" id="text-retroviruses-how-to-stop-them">
<p>
<b>Viruses that have the ability to intergrate into the chromosomes of the
host cell</b>
</p>
</div>

<ol class="org-ol">
<li><a id="early-events"></a>Early Events<br />
<div class="outline-text-5" id="text-early-events">
<ul class="org-ul">
<li>Viruses is uncoated, and uses an enzyme called reverse transcriptase
to turn ssRNA to cDNA, and finally into dsDNA</li>
<li>Then, the enzyme integrase threads the viral dsDNA into the cell's
nucleaus</li>
<li>HIV protease cuts HIV polyproteins into individual parts ready for
budding</li>
</ul>
</div>
</li>

<li><a id="late-events"></a>Late Events<br />
<div class="outline-text-5" id="text-late-events">
<ul class="org-ul">
<li>Proviral region is transcribed slowly whenever ribosome comes across
it by the host DNA polymerase II to make viral proteins + replicate
the viral genome</li>
<li>Components are later exported, assembled, and slowly released through
budding</li>
</ul>

<p>
To make this happen, the virus needs&#x2026;
</p>

<ul class="org-ul">
<li><b>Reverse Transcriptase</b>

<ul class="org-ul">
<li>Transcript RNA to double-stranded RNA</li>
<li>Take double-stranded RNA to turn into DNA</li>
</ul></li>

<li><b>Integrase</b>

<ul class="org-ul">
<li>Force insert the DNA into the genome of the host cell</li>
</ul></li>
</ul>

<p>
And because of the fact that viral DNA is now in cellular DNA, these
viruses' DNAs are hard to get rid of.
</p>

<p>
And this is why we can't cure HIV.
</p>

<p>
Virus, in this case, spread through cell duplication
</p>

<ul class="org-ul">
<li>Proviral region on the DNA, every time the ribosome comes across it,
makes a new viron</li>
<li>These components are then assembled, sent, etc. as usual</li>
<li>Because of the fact that the ribosome needs to, well, come across the
bit of DNA for this to work, the virons are made slowly by âtrickling
out.</li>
</ul>


<div id="org7caaa3d" class="figure">
<p><img src="Screen Shot 2020-10-12 at 11.22.35 PM.png" alt="Screen Shot 2020-10-12 at 11.22.35 PM.png" />
</p>
<p><span class="figure-number">Figure 5: </span>Screen Shot 2020-10-12 at 11.22.35 PM.png</p>
</div>

<p>
<b>Preventing Retroviruses</b>
</p>

<ul class="org-ul">
<li>Prevent Fusion <code>gp120</code>, <code>gp41</code>, <code>CCR5</code></li>
<li>Prevent reverse transcription <code>RT</code></li>
<li>Prevent intergration via intergrease <code>IN</code></li>
<li>Prevent viron maturation <code>PR</code></li>
</ul>


<div id="org9a3f986" class="figure">
<p><img src="stophiv.png" alt="stophiv.png" />
</p>
<p><span class="figure-number">Figure 6: </span>stophiv.png</p>
</div>

<ul class="org-ul">
<li>Most advanced: HAART (Highly-Active Anti-Retroviral Therapy)

<ul class="org-ul">
<li>Cocktail drug works together for inhibition</li>
<li>Two drugs to stop intergration, one to stop protease (viron
maturation)</li>
<li>Could develop resistance</li>
</ul></li>
</ul>
</div>
</li>
</ol>
</div>

<div id="outline-container-viral-genome-vs-mutation-rate" class="outline-4">
<h4 id="viral-genome-vs-mutation-rate"><span class="section-number-4">1.6.6</span> Viral Genome vs Mutation Rate</h4>
<div class="outline-text-4" id="text-viral-genome-vs-mutation-rate">

<div id="org979ee4e" class="figure">
<p><img src="Screen Shot 2020-10-12 at 11.24.39 PM.png" alt="Screen Shot 2020-10-12 at 11.24.39 PM.png" />
</p>
<p><span class="figure-number">Figure 7: </span>Screen Shot 2020-10-12 at 11.24.39 PM.png</p>
</div>

<ul class="org-ul">
<li>RNA viruses could mutate more because it does not have checks</li>
<li>More complex+largest viruses harder to mutate</li>
</ul>

<p>
<b>Genetic drift</b> &#x2014; viruses mutate due to polymerase error
</p>

<p>
<b>Genetic shift</b> &#x2014; viruses recombinate without mutating by
crossing-over mechanism or genome segment reassortment. Think! the flu
</p>
</div>
</div>
</div>

<div id="outline-container-why-are-viruses-bad" class="outline-3">
<h3 id="why-are-viruses-bad"><span class="section-number-3">1.7</span> Why are viruses bad</h3>
<div class="outline-text-3" id="text-why-are-viruses-bad">
<p>
Damage host cells/tissues by&#x2026;
</p>

<ul class="org-ul">
<li>Reducing gene expression capacity</li>
<li>Depleting cellular resources</li>
<li>Causing cell lysis (to explode)</li>
<li>Promoting tumorigenisis &#x2014; cancer</li>
<li>Creating damaging immunological response</li>
</ul>
</div>
</div>

<div id="outline-container-preventing-viruses" class="outline-3">
<h3 id="preventing-viruses"><span class="section-number-3">1.8</span> Preventing Viruses</h3>
<div class="outline-text-3" id="text-preventing-viruses">
<p>
Let's talk about <b>Remdesivir</b>! A drug developed by Pfizer that's used to
combat Ebola + influenza viral replication.
</p>

<p>
Modified nucleotide triphosphate which adds onto the RNA strand copied
by the RNA-Dependent RNA Polymerase carried by viruses
</p>

<ul class="org-ul">
<li>Pretends + gets inserted as a nucleotide</li>
<li>Once added onto the RNA chain, jams further actual nucleotides from
being inserted</li>
</ul>

<p>
<i>Could</i> but usually does not jam up normal RNA polymerase which does
normal transcription
</p>

<ul class="org-ul">
<li>Inhibiting transcription in the short term won't kill you immediately</li>
<li>So, we hurt normal cell transcription a little in order to rid of the
virus</li>
<li>Need hospital treatment for regular and safe dosing for this exact
reason</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-cn10212020" class="outline-2">
<h2 id="cn10212020"><span class="section-number-2">2</span> CN10212020</h2>
<div class="outline-text-2" id="text-cn10212020">
</div>
<div id="outline-container-making-proteins-a-guide" class="outline-3">
<h3 id="making-proteins-a-guide"><span class="section-number-3">2.1</span> Making Proteins, a guide</h3>
<div class="outline-text-3" id="text-making-proteins-a-guide">
<p>
<b>Genetic Code</b> =&gt; "nucleotide code" found in the DNA that helps make
protein. There are two parts of this: translation and transcription.
</p>

<ul class="org-ul">
<li>The process of <b>Transcription</b> involves taking the DNA, separating it,
and copying its corresponding pairs to RNA</li>
<li>The process of <b>Translation</b> involves taking the RNA and making
proteins.</li>
</ul>

<p>
Occasionally, the RNA is what we want to end up with, so then obviously
we no longer need the process of Translation.
</p>
</div>

<div id="outline-container-transcription-converting-dna-to-mrna" class="outline-4">
<h4 id="transcription-converting-dna-to-mrna"><span class="section-number-4">2.1.1</span> Transcription =&gt; converting DNA to mRNA</h4>
<div class="outline-text-4" id="text-transcription-converting-dna-to-mrna">
<ul class="org-ul">
<li>Done by RNA Polymerase Enzyme</li>
<li>Rip apart hydrogen bonds using DNAse enzyme</li>
<li>Read one side ("template strand", a.k.a. noncoding strand) of the
double helix, recognizing each nucleotide</li>
<li>Pluck the correct corresponding nucleotide out of the nucleus

<ul class="org-ul">
<li>G-&gt;C</li>
<li>C-&gt;G</li>
<li>A-&gt;*U*</li>
<li>T-&gt;A</li>
</ul></li>

<li>Prokaryotes lack membrane-bound nucleus (or any organelle)</li>
</ul>

<p>
\definition{Gene}{information that successfully encodes a functional protein or a functional catalytic RNA}
RNAs could also be catalysts!
</p>

<ul class="org-ul">
<li>"Promoter"s denotes beginning of a gene. "Terminator"s denotes the end
of gene.</li>
</ul>

<p>
<b>Starting Transcription</b> * Series of utility "factors" proetins begin to
assemble to call the attention of RNA polymerase. (#how + #when does
this happen? #ASK) * RNA polyamerase binds to the Sigma Subunit =&gt; form
a holoenzyme to unwind DNA * Sigma subunit informs the enzyme where to
find a promoter (beginning of binding) * "Enhancer" gene sequences help
bind with activator proteins to help attract RNA polymerase II
</p>

<p>
<b>Promoters</b>
</p>

<ul class="org-ul">
<li>Polymerase Enzyme starts at a promoter (typically found upstream of
the 5' start site) and ends at a terminator

<ul class="org-ul">
<li>Box of TATTAA highlights transcription rate and the start site</li>
<li>TFIIA cofactor in RNA recognizes TATTAA box, TFIIB recognizes
C/CG/CG/CGCCC upstream</li>
</ul></li>

<li>Stronger promoters/enhancers =&gt; "enhance" "more." i.e.Â tumor viruses
strengthen promoters for cell growth</li>
</ul>

<p>
<b>Terminators</b>
</p>

<ul class="org-ul">
<li>Found in the end of the template sequence</li>
<li>Two types in prokaryotes

<ul class="org-ul">
<li>Rho-independent terminators &#x2014; roll back onto itself, causing the
RNA to terminate and mRNA to be release</li>
<li>Rho-dependent terminators &#x2014; activate cofactor named rho + unwind
the transcribed RNA-DNA hybrid</li>
</ul></li>

<li>In Eukarotes

<ul class="org-ul">
<li>Pol I genes &#x2014; transcription stopped through termination factor by
unwindng the transcribed RNA-DNA hybrid</li>
<li>Pol II genes &#x2014; don't stop until the end, but a polymerase has a
"cleavage" mechanism that clips the end out using a poly(A) tail
consensus sequence</li>
</ul></li>
</ul>
</div>
</div>

<div id="outline-container-before-we-continue-two-words" class="outline-4">
<h4 id="before-we-continue-two-words"><span class="section-number-4">2.1.2</span> Before we continue, two words</h4>
<div class="outline-text-4" id="text-before-we-continue-two-words">
<ul class="org-ul">
<li><i>Non-coding sequence</i>: metadata for DNA for the processors</li>
<li><i>Coding sequence</i>: DNA content for amino-acid production</li>
</ul>
</div>
</div>

<div id="outline-container-mrna-processing-splicing-mrna" class="outline-4">
<h4 id="mrna-processing-splicing-mrna"><span class="section-number-4">2.1.3</span> mRNA processing =&gt; splicing mRNA</h4>
<div class="outline-text-4" id="text-mrna-processing-splicing-mrna">
<p>
Pre-process the mRNA.
</p>

<p>
<b>Prokaryotes does not do this!</b> Prokarotes' coding sequence always makes
a full protein, so we just start at promoter and end at terminator and
make a protein!
</p>

<p>
In Eukaryotic DNA&#x2026;
</p>

<p>
Between Promoter and Terminator, <b>Exon</b> and <b>Intron</b> alternate. Exon is
coding, whereas Intron is non-coding and works as metadata.
</p>

<p>
After reading the intron, they are spliced out during mRNA processing =&gt;
done by the "splicesome". The mRNA, after splicing, is "capped and
tailed" to mark pre-processing completion, at which point they leave the
nucleus + go to the ribosome.
</p>

<ul class="org-ul">
<li>Begin by assembling helper proteins at intron-exon borders =&gt; "slicing
factors"</li>
<li>Other helping factor proteins come together and form the "splicesome"
to do the splicing</li>
<li>Splicesome splices by bringing exon ends together</li>
<li>After it's done, the splicesome disintergrates</li>
</ul>
</div>
</div>

<div id="outline-container-translation-rna-directed-polypeptide-synthesis" class="outline-4">
<h4 id="translation-rna-directed-polypeptide-synthesis"><span class="section-number-4">2.1.4</span> Translation =&gt; RNA-directed polypeptide synthesis</h4>
<div class="outline-text-4" id="text-translation-rna-directed-polypeptide-synthesis">
<p>
Mature mRNA sent to ribosome. mRNA must travel to the cytoplasm in the
Eukarotes to catch the RNA, whereas in prokarotes they don't have to go
anywhere.
</p>

<p>
Ribosomes has two units: 50S unit + 30S unit =&gt; they come together
whenever a mRNA needs it. Each contained specialized rRNA + tTRNA to
catalyze attachment of and carry amino acids + adapt the incoming mRNA
respectively.
</p>

<p>
<b>Note! The beginning of mRNA is not translated.</b> There a portion on the
5' end of the mRNA (starts with AGGAGG) &#x2014; about 170 nuclotides in
humans, and shorter in bacteria &#x2014; that's called UTR (untranslated
region.) This region helps ribosomes bind to it + stablize the binds.
</p>

<ul class="org-ul">
<li>3 protein factors IF1, IF2, IF3 forms a complex for transcription by
binding to a subunit on the ribosome</li>
<li>Methionine-carrying tRNA binds to the start of the mRNA, which forms
the initiation complex. This is typically removed after translation if
not coded for (f M-A amino acid pair coded for, methonine removed; but
if M-L pairs coded for, methonine not removed.)</li>
<li>A-site: translates mRNA to tRNA &#x2014; anti-codon pairs</li>
<li>P-site: amino acid dumped from tRNA to the actual chain being built</li>
<li>Spent tRNA ejected to the E-site, which is then recycled</li>
<li>Catalyst tRNA combines with rRNA to catalyze amino acid peptide bond</li>
<li>Each codon (group of 3 units in tRNA), matches a specific
<a href="KBhBIO101AminoAcids.html">KBhBIO101AminoAcids</a></li>
</ul>

<p>
Smaller ribosome unit grabs, larger attaches + forms amino acid
</p>

<p>
After the amino acids are assembled, it's time for
<a href="KBe2020bio101refProteinFolding.html">KBe2020bio101refProteinFolding</a>.
See also <a href="KBhBIO101Proteins.html">KBhBIO101Proteins</a>.
</p>

<p>
=&gt; Shaperones fold proteins, and if its finds proteins impossible to
fold, it flags it using ubiquitin to send to the garbage
</p>

<hr />

<p>
Eukarotic gene expression is regulated at many stages &#x2014; prevents error
</p>


<div id="orga6e6f5e" class="figure">
<p><img src="preprocessing.png" alt="preprocessing.png" />
</p>
<p><span class="figure-number">Figure 8: </span>preprocessing.png</p>
</div>

<ul class="org-ul">
<li>Viral proteins are usually easy to assemble</li>
</ul>


<div id="org556d299" class="figure">
<p><img src="Screen Shot 2020-11-02 at 2.48.22 PM.png" alt="Screen Shot 2020-11-02 at 2.48.22 PM.png" />
</p>
<p><span class="figure-number">Figure 9: </span>Screen Shot 2020-11-02 at 2.48.22 PM.png</p>
</div>

<p>
In Prokarotes, there are less steps.
</p>

<p>
Between the promoter and the actual coding DNA, there is a region named
<i>operator</i> that allows three types of regulatory molecules to bind to it
to alter how the gene is transcribed, namely:
</p>

<ul class="org-ul">
<li><b>Repressors</b>: proteins that suppress transcription</li>
<li><b>Activators</b> are proteins that increase the transcription</li>
<li><b>Inducers</b> catalyses repressors or activators &#x2014; making either a
strenthened activation or repression acting in conjunction with the
other regulator</li>
</ul>

<p>
<b>Question: how are proteins made in the viral genome</b>
</p>

<ul class="org-ul">
<li>No viruses produce ribosomes</li>
<li>Ribosomes become centrally important for the virus</li>
<li>What serves as the template to make new virus copies</li>
</ul>

<p>
Viruses attempt to overwhelm the enzyme to entry.
</p>

<ul class="org-ul">
<li><b>DNA Polymerase</b> takes DNA and makes more DNA

<ul class="org-ul">
<li>Duplicates cell DNA</li>
<li>Could be hijacked during cell cycle to duplicate DNA viruses</li>
<li>DNA viruses may also carry their DNA Polymerease to not wait for the
cell cycle</li>
</ul></li>

<li><b>RNA Polymerease</b> takes DNA and makes mRNA

<ul class="org-ul">
<li>Have lower fidelity with an error about 1/100,000</li>
<li>Hence why safety mechanism needed</li>
</ul></li>
</ul>

<p>
<b>Capping and Tailing</b> * 3' end =&gt; AAAAAA tail (using poly-adenine
tailing enzyme) * 5' end =&gt; GGGGGG cap (using guanine-capping enzyme)
</p>

<p>
<b>DNA</b> viruses are "less complex", in that as long as they are able to
get into the nucleaus, the rest would just be the body's work
automatically.
</p>
</div>
</div>

<div id="outline-container-list-of-kool-proteins" class="outline-4">
<h4 id="list-of-kool-proteins"><span class="section-number-4">2.1.5</span> List of Kool Proteins</h4>
<div class="outline-text-4" id="text-list-of-kool-proteins">
<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">


<colgroup>
<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left">Name</th>
<th scope="col" class="org-left">Function</th>
</tr>
</thead>
<tbody>
<tr>
<td class="org-left">RNA Polymerase</td>
<td class="org-left"><i>transcripts</i>: takes DNA and turns into mRNA</td>
</tr>

<tr>
<td class="org-left">DNA Polymerase</td>
<td class="org-left"><i>replicates</i>: takes DNA and makes more copy of it</td>
</tr>

<tr>
<td class="org-left">RNA-Dependent RNA Polymerase</td>
<td class="org-left"><i>replicates</i>: takes RNA and makes more copy of it. Basically only viruses use it.</td>
</tr>

<tr>
<td class="org-left">Promoter</td>
<td class="org-left"><i>signals</i>: DNA signal of the start of the DNA.</td>
</tr>

<tr>
<td class="org-left">Terminator</td>
<td class="org-left"><i>signals</i>: DNA signal of the end of the DNA.</td>
</tr>
</tbody>
</table>

<hr />
</div>
</div>
</div>

<div id="outline-container-dna-replication-and-the-cell-cycle" class="outline-3">
<h3 id="dna-replication-and-the-cell-cycle"><span class="section-number-3">2.2</span> DNA Replication and the Cell Cycle</h3>
<div class="outline-text-3" id="text-dna-replication-and-the-cell-cycle">
</div>
<div id="outline-container-so-why-do-cell-divide" class="outline-4">
<h4 id="so-why-do-cell-divide"><span class="section-number-4">2.2.1</span> So, why do cell divide</h4>
<div class="outline-text-4" id="text-so-why-do-cell-divide">
<blockquote>
<p>
The ability to produce organisms more of their kind is one
characteristic that best distinguishes living things from nonliving
matter
</p>
</blockquote>

<p>
Viruses + Organelles challenge this definition =&gt; they are symbiotic and
cannot reproduce on their own. We tend to think that cells
</p>

<ul class="org-ul">
<li>Everyday, 50-70 Billion die =&gt; <b>programmed cell death</b></li>
<li>To compensate this, Mitosis (cell division) happen

<ul class="org-ul">
<li>Cell divide in opposite directions</li>
<li>Two strands ANTIPARALLEL to each other</li>
</ul></li>
</ul>
</div>
</div>

<div id="outline-container-levels-of-dna-structure" class="outline-4">
<h4 id="levels-of-dna-structure"><span class="section-number-4">2.2.2</span> Levels of DNA structure</h4>
<div class="outline-text-4" id="text-levels-of-dna-structure">
<ul class="org-ul">
<li>DNA Double Helix =&gt; Unwrapped, raw DNA</li>
<li>Histones =&gt; coiler proteins at which DNA coils around

<ul class="org-ul">
<li>Four histones</li>
<li>H3, H4, H2a, H2b wraps, H1 seals the four together</li>
<li>Each protein has 1.65 wraps</li>
</ul></li>

<li>Nucleosomes =&gt; stack of 2 histone groups to create a spool of 8
histone proteins wrapping DNA</li>
<li>Nucleosomes wraps around into a large fiber called "chromatin"</li>
<li>A pair of chromatin entangle to form a chromasome</li>
</ul>
</div>
</div>

<div id="outline-container-cell-divison-process" class="outline-4">
<h4 id="cell-divison-process"><span class="section-number-4">2.2.3</span> Cell divison process</h4>
<div class="outline-text-4" id="text-cell-divison-process">
<ul class="org-ul">
<li>Chromesomes line up in equator</li>
<li>Each chromesome has two chromatid exactly the same</li>
<li>Microtubials to pull chromesomes appart connected to kinecore, a joint
in the chromatid</li>
<li>Kinetore senses tension, and when it is correct, molecules are sent
down the microtubials to send a split signal</li>
</ul>


<div id="org4c2e9ed" class="figure">
<p><img src="levelsofdna.png" alt="levelsofdna.png" />
</p>
<p><span class="figure-number">Figure 10: </span>levelsofdna.png</p>
</div>


<div id="org4dadbaa" class="figure">
<p><img src="histones.png" alt="histones.png" />
</p>
<p><span class="figure-number">Figure 11: </span>histones.png</p>
</div>

<p>
<b>Most cell division results in genetically identical daughter cell</b>
</p>

<p>
Each cell, once specialised, chooses what parts of their chromasome to
unwrap + permanently wrap.
</p>

<p>
Difference in transcription results in different phenotypes.
</p>

<p>
Sperm + Egg (imcomplete cells) combine together to form a "zygote" =&gt; a
single cell. Each person is from a zygote.
</p>

<p>
For Eukarotes, cells divide using Mytosis.
</p>

<p>
<a href="https://docs.google.com/document/d/1TIrgR9VSV3attTK_QP-AOCs33mMoBP0Cz7DQXysKoD0/edit">Paul's
Cell Cycle Primer</a>
</p>


<div id="orgc6ffc47" class="figure">
<p><img src="Screen Shot 2020-11-09 at 3.16.12 PM.png" alt="Screen Shot 2020-11-09 at 3.16.12 PM.png" />
</p>
<p><span class="figure-number">Figure 12: </span>Screen Shot 2020-11-09 at 3.16.12 PM.png</p>
</div>
</div>
</div>

<div id="outline-container-major-cell-cycle-parts" class="outline-4">
<h4 id="major-cell-cycle-parts"><span class="section-number-4">2.2.4</span> Major Cell Cycle Parts</h4>
<div class="outline-text-4" id="text-major-cell-cycle-parts">
<ul class="org-ul">
<li>G1 =&gt; Rest Phase, Gap 1

<ul class="org-ul">
<li>May hit s.a. to volume checkpoint =&gt; if ratio too big, the cell is
too big</li>
<li>May hit diffusion checkpoint =&gt; larger cells would need to work
harder to transport things to the centre</li>
</ul></li>

<li>S =&gt; S Phase, duplicate DNA. 150 mins</li>
<li>G2 =&gt; Rest Phrase, Gap 2. The pairs of DNA begins bundling and
condensing; the DNA is also checked upon and verified for consistency</li>
<li>M =&gt; Mitosis!

<ul class="org-ul">
<li>Prophase: condensation of chromasome</li>
<li>Chromatin aligns in the center =&gt; (metaphase)</li>
<li>A divit (DEFINITION?) forms</li>
<li>Two chromatin becomes pulled apart =&gt; forming two cells via
cytokenisis (anephase)</li>
</ul></li>
</ul>

<p>
Cell regulators are proteins that manage and sheperard the process of
cell division. They respond to molecular signals throughout the cell and
check for internal signals like DNA damage to control the rate and
progress of cell division.
</p>


<div id="orga8acb95" class="figure">
<p><img src="lecellcycle.png" alt="lecellcycle.png" />
</p>
<p><span class="figure-number">Figure 13: </span>lecellcycle.png</p>
</div>
</div>
</div>

<div id="outline-container-dna-replication-process" class="outline-4">
<h4 id="dna-replication-process"><span class="section-number-4">2.2.5</span> DNA Replication Process</h4>
<div class="outline-text-4" id="text-dna-replication-process">
<p>
DNA replication is known to be "semi-conservative" &#x2014; meaning that it
is a process that pairs a synthesized half of the DNA with an original
half of the DNA (i.e.Â takes the ORIGINAL template strand + makes the NEW
coding strand &amp; takes the ORIGINAL coding strand + makes the NEW
template strand.)
</p>

<p>
Because <b>polymerases copy uni-directionally</b> =&gt; DNA polyemrease move
along the 3' to 5' DNA to create a copy 5' to 3'. Meaning, the
polymerize is able to add nucleotide onto the 3' end of the DNA.
</p>

<ul class="org-ul">
<li>Open the DNA at an arbuiturary point using the Helicase

<ul class="org-ul">
<li>Uses two helicase =&gt; one open rightward, and one leftward. The
movement of the helicase opening the DNA is called the "fork
movement"</li>
<li>DNA polymerase could only add nucleotides 5' to 3'</li>
<li>As helicase open a little bit of the DNA, polymerases rushes to copy
the area that opened

<ul class="org-ul">
<li>In the <b>leading</b> strand (3' to 5'), polymerase will run alongside
the helicase for they are opening and replicating on the same
direction</li>
<li>In the <b>lagging</b> strand (5' to 3'), polymerase will wait until the
helicate opens a little segment, and rushes forward and move
backwards

<ul class="org-ul">
<li>NOTE: the lagging strand&#x2026; 1) takes longer to transcribe 2) is
done in small chunks (each "rush forward"). Each chunk is called
an ogazaki fragment</li>
</ul></li>
</ul></li>
</ul></li>
</ul>


<div id="orgeba31e9" class="figure">
<p><img src="leadinglagging.png" alt="leadinglagging.png" />
</p>
<p><span class="figure-number">Figure 14: </span>leadinglagging.png</p>
</div>

<ul class="org-ul">
<li>DNA polymersease will REQUIRE a double-stranded area to begin work
from, so Primase synthesize already double-stranded RNA primers that
DNA polymerease could bootstrap to the single-stranded DNA to begin
the replication process (think: create-react-app)</li>

<li>DNA polymerse will detect unfitting bonds and remove leftover RNA
primer bootstrap units to repair them in a process called
"proofreading." DNA polimersease is assisted with "glue" ligase to
help the DNA polymerease pick out and replace problematic/unneeded
nucleotides and perhaps their neighbors. This is where the Ogazaki
fragments get joined.</li>
</ul>

<p>
Steps of DNA replication, in Paul's words:
</p>

<pre class="example" id="org7fa8a0b">
- Many proteins work together in DNA replication and repair. 
The process of DNA replication is semiconservative, such that takes place through complementary base pairing of a template strand of parent DNA. 

- The process of replication begins at the origin of replication, forming a replication fork. The enzyme, helicase, unwinds DNA exposing template strands, primase synthesizes RNA primers to begin the process

- Topoisomerase breaks, swivels, and rejoins the parent DNA to relieve strain caused by unwinding. 

- DNA polymerase is the enzyme that catalyzes the process of complementary base pairing of nucleotides to the template strand. 

- New nucleotide strands always for in the 5â to 3â direction, therefore the leading strand forms continuously but the lagging strand is formed in Okazaki fragments (still in the 5â to 3â direction) and connected by the enzyme ligase. 

- DNA polymerase is able to proofread pairing, and along with mismatch repair enzymes, DNA is carefully check and repair DNA. 

- The end of a DNA molecule are called telomeres (not in circular genome e.g. bacteria), and shorten during each replication (Hayflick limit). Noncoding, repeating units of nucleotides act as protection from losing essential genetic information by shorting. 
</pre>
</div>
</div>
</div>
</div>
</div>
<div id="postamble" class="status">
<div class="namedate"><p class="subtitle"><span class="author">Houjun Liu</span><span class="date">2021-09-11 Sat 09:29</span></p></div><div class="taprootcopyright">(c) 2021 Houjun Liu, Huxley Marvit, Zachary Sayyah, Albert Huang, Dylan Wallace</div>
</div>
</body>
</html>
