<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2021-09-11 Sat 10:10 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Single Nucleotide Polymorphism</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Houjun Liu" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: auto;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline; margin-top: 14px;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" href="https://taproot3.sanity.gq/global.css" type="text/css"/>
<script type="text/javascript">
// @license magnet:?xt=urn:btih:e95b018ef3580986a04669f1b5879592219e2a7a&dn=public-domain.txt Public Domain
<!--/*--><![CDATA[/*><!--*/
     function CodeHighlightOn(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.add("code-highlighted");
         target.classList.add("code-highlighted");
       }
     }
     function CodeHighlightOff(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.remove("code-highlighted");
         target.classList.remove("code-highlighted");
       }
     }
    /*]]>*///-->
// @license-end
</script>
</head>
<body>
<div id="org-div-home-and-up">
 <a accesskey="h" href="https://taproot3.sanity.gq/sitemap.html"> UP </a>
 |
 <a accesskey="H" href="https://taproot3.sanity.gq/theindex.html"> HOME </a>
</div><div id="content">
<h1 class="title">Single Nucleotide Polymorphism</h1>


<div id="outline-container-snp" class="outline-2">
<h2 id="snp"><span class="section-number-2">1</span> SNP</h2>
<div class="outline-text-2" id="text-snp">
<p>
<b>Single Nucleotipe Polymorphism</b>: pairs of single-base pair variations
of genes that are not uncommon (&gt;1%), and hence not quite considered a
mutation.
</p>

<p>
SNPs could be in any part of the genome: sometimes in a gene, but more
often in a non-coding region.
</p>


<div id="org1b04f3e" class="figure">
<p><img src="./Pasted image 20210505134813.png" alt="Pasted image 20210505134813.png" />
</p>
<p><span class="figure-number">Figure 1: </span>Pasted image 20210505134813.png</p>
</div>

<p>
These could do absolutely nothing, but they could also create large
trait differences (sickle-cell is a SNP).
</p>

<p>
<i>The DNA by mail kits are actually just looking for SNPs!</i>
</p>

<p>
Multiple SNPs come together to form a "Haplotype"; they are SNPs that
are fairly close together that frequently occur in sequence (like the
chain of three SNPs A something something G something something T would
be a haplotype.)
</p>

<p>
People of the same haplotype are likely of same ancestry.
</p>
</div>

<div id="outline-container-snp-guidelines" class="outline-3">
<h3 id="snp-guidelines"><span class="section-number-3">1.1</span> SNP Guidelines</h3>
<div class="outline-text-3" id="text-snp-guidelines">
<ul class="org-ul">
<li>No medical SNPs; so avoid SNPs that could be associated with heavy
medical information</li>
<li>No SNPs that could potentially give you PTSD</li>
</ul>
</div>
</div>

<div id="outline-container-the-reaction" class="outline-3">
<h3 id="the-reaction"><span class="section-number-3">1.2</span> the Reaction</h3>
<div class="outline-text-3" id="text-the-reaction">
<ul class="org-ul">
<li>Each student's PCR reaction will only amplify 500-800 nucleotide
region</li>
<li>Find SNPs that have specific traits independently, because most SNPs
are polygenic (they tell you info only if you discover many different
SNPs, which is complicated and hard)</li>
<li>SNPs are sometimes discovered using Genome Wide Association Studies,
which is challenging to deal with because they may be flawed</li>
</ul>

<hr />
</div>
</div>

<div id="outline-container-polymerase-chain-reaction-pcr" class="outline-3">
<h3 id="polymerase-chain-reaction-pcr"><span class="section-number-3">1.3</span> Polymerase Chain Reaction (PCR)</h3>
<div class="outline-text-3" id="text-polymerase-chain-reaction-pcr">
<p>
<i>A reaction by which the DNA replication process is attempted to be
replicated out of the cell in preparation to sequence one's DNA</i>
</p>

<p>
<b>PCR provides a way of creating <i>lots</i> of copies of a <i>small</i> strand of
DNA</b>
</p>
</div>

<div id="outline-container-reaction-need" class="outline-4">
<h4 id="reaction-need"><span class="section-number-4">1.3.1</span> Reaction Need</h4>
<div class="outline-text-4" id="text-reaction-need">
<ul class="org-ul">
<li>The DNA Portion (duh)</li>
<li>A buffer to host the liquid</li>
<li>Primers - Bit of RNA to help DNA polymerase get going</li>
<li>DNA Polymerase - The Enzyme to Copy the DNA =&gt; Typically, the TAQ
Polymerease is used because it is from a bacterial that's quite
resistant to heat</li>
<li>DNA Nucleotides - to build the replicated DNA</li>
</ul>
</div>
</div>

<div id="outline-container-steps" class="outline-4">
<h4 id="steps"><span class="section-number-4">1.3.2</span> Steps</h4>
<div class="outline-text-4" id="text-steps">
<ol class="org-ol">
<li>Denaturation

<ol class="org-ol">
<li>Cook the DNA ("denature it"/heat it) to seperate the two strands
of the DNA</li>
</ol></li>

<li>Annealing

<ol class="org-ol">
<li>Cool the Cooked DNA, and dump in the primers</li>
<li>The specific temperature in this step is important because it
controlls where the primer binds, and hence which part of the DNA
you want to amplify</li>
</ol></li>

<li>DNA Synthesis =&gt; make more copies of the DNA

<ol class="org-ol">
<li>This needs a slightly higher temperature than annealing, but which
one depends on what polymerase you used</li>
</ol></li>

<li>Go to step 1</li>
</ol>

<p>
Note: the DNA polymerase may run over, so we just keep repeating this
and it will slowly become more successful as the DNA polymerease repeats
its work repeatedly.
</p>

<p>
BTW: if you read carefully, you will realize that you made TWO DNA
double-strands out of a single double strand. So exponential growth
baybee and then you could get lots of copies of that region.
</p>
</div>
</div>

<div id="outline-container-but-why" class="outline-4">
<h4 id="but-why"><span class="section-number-4">1.3.3</span> But, why?</h4>
<div class="outline-text-4" id="text-but-why">
<p>
In order to have enough copies for gel electroprorisis (like, the
rRT-PCR COVID test) or, for things like crime forensics, you need many
copies of the same DNA for a clearly detectable result for analysis that
may have a high threshold.
</p>
</div>
</div>
</div>
</div>
</div>
<div id="postamble" class="status">
<div class="namedate"><p class="subtitle"><span class="author">Houjun Liu</span><span class="date">2021-09-11 Sat 09:29</span></p></div><div class="taprootcopyright">(c) 2021 Houjun Liu, Huxley Marvit, Zachary Sayyah, Albert Huang, Dylan Wallace</div>
</div>
</body>
</html>
