<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2021-09-11 Sat 16:37 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Eukaryotic Gene Expression</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Zachary Sayyah" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: auto;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline; margin-top: 14px;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" href="https://taproot3.sanity.gq/global.css" type="text/css"/>
<script type="text/javascript">
// @license magnet:?xt=urn:btih:e95b018ef3580986a04669f1b5879592219e2a7a&dn=public-domain.txt Public Domain
<!--/*--><![CDATA[/*><!--*/
     function CodeHighlightOn(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.add("code-highlighted");
         target.classList.add("code-highlighted");
       }
     }
     function CodeHighlightOff(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.remove("code-highlighted");
         target.classList.remove("code-highlighted");
       }
     }
    /*]]>*///-->
// @license-end
</script>
</head>
<body>
<div id="org-div-home-and-up">
 <a accesskey="h" href="https://taproot3.sanity.gq/sitemap.html"> UP </a>
 |
 <a accesskey="H" href="https://taproot3.sanity.gq/theindex.html"> HOME </a>
</div><div id="content">
<h1 class="title">Eukaryotic Gene Expression</h1>


<div id="outline-container-flo-ret" class="outline-3">
<h3 id="flo-ret"><span class="section-number-3">0.1</span> #flo #ret</h3>
<div class="outline-text-3" id="text-flo-ret">
</div>
</div>
<div id="outline-container-notes" class="outline-2">
<h2 id="notes"><span class="section-number-2">1</span> Notes</h2>
<div class="outline-text-2" id="text-notes">
</div>
<div id="outline-container-overview" class="outline-4">
<h4 id="overview"><span class="section-number-4">1.0.1</span> Overview</h4>
<div class="outline-text-4" id="text-overview">
<ul class="org-ul">
<li>Organisms turn genes on and off which is called Gene Expression

<ul class="org-ul">
<li>This can be done in response to external and internal signals

<ul class="org-ul">
<li>These signals are based off of environmental factors</li>
</ul></li>

<li>This is also be done in order to specialize cells

<ul class="org-ul">
<li>Certain cells need certain genes to preform their specific role</li>
</ul></li>
</ul></li>
</ul>

<p>
### Differential Gene Expression
</p>

<ul class="org-ul">
<li>Human Cells can express about 20% of it's protein coded genes at any
given time</li>
<li>Most cells contain the same genome

<ul class="org-ul">
<li>Each cell type must use specific parts of this genome

<ul class="org-ul">
<li>This is called Differential gene expression</li>
</ul></li>

<li>Exception would be cells of the immune system</li>
</ul></li>

<li>Due to the importance of gene expression when it has issues it can
affect the organism significantly</li>
<li>Process of Gene expression in a Eukaryotic cell

<ul class="org-ul">
<li>Chromatin (DNA unpacking) &#x2013;&gt;</li>
<li>RNA processing &#x2013;&gt;</li>
<li>Transport to cytoplasm &#x2013;&gt;</li>
<li>Translation &#x2013;&gt;</li>
<li>Protein processing &#x2013;&gt;</li>
<li>Transport to cellular destination&#x2013;&gt;</li>
</ul></li>

<li>This process can often be equated to transcription for Prokaryote
cells</li>
</ul>
</div>
</div>

<div id="outline-container-regulation-of-chromatin-structure" class="outline-4">
<h4 id="regulation-of-chromatin-structure"><span class="section-number-4">1.0.2</span> Regulation of Chromatin Structure</h4>
<div class="outline-text-4" id="text-regulation-of-chromatin-structure">
<ul class="org-ul">
<li>The chromatin structure itself allows for the regulation of gene
expression

<ul class="org-ul">
<li>This is partially due to the location of the promoter</li>
</ul></li>

<li>Chemical modifications to the histone proteins can affect the
structure

<ul class="org-ul">
<li>This in turn can affect gene expression</li>
<li>Histone proteins are the proteins in which the DNA is wrapped</li>
<li>There are many types of modifications that can take place

<ul class="org-ul">
<li>Histone acetylation can tend to promote transcriptions by opening
up the chromatin</li>
<li>Additional methyl groups tend to close up the chromatin and
decrease transcription</li>
</ul></li>

<li>DNA methylation occurs in most plants and animals as well as fungi</li>
<li>Methylated DNA will stay methalated through cell divisions

<ul class="org-ul">
<li>This accounts for genomic imprinting</li>
<li>These epigenetic markers can be inherited

<ul class="org-ul">
<li>There is continually more evidence for the importance of
epigentics in gene expression ### Regulation of Transcription</li>
</ul></li>
</ul></li>
</ul></li>

<li>Chromatin changes are not permanent and can be reversed</li>
<li>The next step of gene expression regulation is in the transcription
factors

<ul class="org-ul">
<li>These either allow for or inhibit transcription</li>
</ul></li>

<li>These factors usually bind to proteins, but some of them bind to DNA</li>
<li>High levels of transcription factors created for specific genes are
associated with another protein thought creatively of as specific
transcription factors</li>
<li>Gene expression is dramatically increased or decreased by the binding
of specific transcription factors

<ul class="org-ul">
<li>These are either activators or repressors</li>
</ul></li>

<li>There are many transcription factors

<ul class="org-ul">
<li>Repressors act in many different ways, but some bind directly to
control element DNA blocking activator binding</li>
<li>Others interfere with the activator itself</li>
</ul></li>

<li>Coordinated control of genes can need to happen when multiple genes
need to be expressed at the same time for something to function

<ul class="org-ul">
<li>These can often be signaled from the outside with something like a
hormone</li>
</ul></li>

<li>The activation of receptors on the surface of the cell can release
specific repressors and activators</li>
</ul>

<p>
### Mechanisms of Post-Transcriptional Regulation - Transcription is not
the only thing that regulates gene expression - How much of the protein
is created once the RNA is received is also a factor - RNA can be
interpreted in different ways with different things being introns and
others being exons - This allows for the creation of multiple proteins
from the same strand of RNA - RNA splicing is critical since it allows a
lot of information to be fit on a single strand of RNA - Around 75-100%
of human genes with multiple exons undergo RNA splicing allowing for our
genome to describe a lot of complexity without needing as many genes -
Translation is another stage at which gene expression occurs - Some
regulatory proteins can bock translation of an mRNA by preventing
attachment to a ribisome - Length by which an mRNA is around is also
crucial - This can vary greatly depending on the cell - Cells can mark
proteins for destruction using something called ubiquitin
</p>
</div>
</div>
</div>
</div>
<div id="postamble" class="status">
<div class="namedate"><p class="subtitle"><span class="author">Zachary Sayyah</span><span class="date">2021-09-11 Sat 16:35</span></p></div><div class="taprootcopyright">(c) 2021 Houjun Liu, Huxley Marvit, Zachary Sayyah, Albert Huang, Dylan Wallace</div>
</div>
</body>
</html>
