<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2021-10-19 Tue 10:08 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Fall Project 1 Master Doc</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Exr0n" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: auto;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline; margin-top: 14px;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" href="/global.css" type="text/css"/>
<link rel="stylesheet" href="/admonition.css" type="text/css"/>
<script type="text/javascript">
// @license magnet:?xt=urn:btih:e95b018ef3580986a04669f1b5879592219e2a7a&dn=public-domain.txt Public Domain
<!--/*--><![CDATA[/*><!--*/
     function CodeHighlightOn(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.add("code-highlighted");
         target.classList.add("code-highlighted");
       }
     }
     function CodeHighlightOff(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.remove("code-highlighted");
         target.classList.remove("code-highlighted");
       }
     }
    /*]]>*///-->
// @license-end
</script>
</head>
<body>
<div id="preamble" class="status">
<div class="header"><span class="site"><a href="https://taproot3.sanity.gq">TR3.5</a></span></div><div class="datarow"><h1 class="title">Fall Project 1 Master Doc</h1> <h2 class="subtitle"></h2> <span class="author">Exr0n</span> <span class="date">2021-10-19 Tue 10:08</date></div>
</div>
<div id="content">
<div id="linktables"><div id="table-of-backlinks"><h2>Backlinks</h2><ul></ul></div><div id="table-of-contents"><h2>Table of Contents</h2><div id="text-table-of-contents">
<ul>
<li><a href="#org00b9a85">1. Preliminary Research</a>
<ul>
<li><a href="#org7e13db1">1.1. Sources</a></li>
</ul>
</li>
<li><a href="#org04077bc">2. Notes</a>
<ul>
<li><a href="#org5fd3bb5">2.1. Target Processes</a>
<ul>
<li><a href="#org555952b">2.1.1. Enzyme catalysis</a></li>
<li><a href="#orge669a25">2.1.2. Protein-ligand binding</a></li>
<li><a href="#org29b1964">2.1.3. signal transduction</a></li>
<li><a href="#orgdd3d8aa">2.1.4. allosteric regulation</a></li>
</ul>
</li>
<li><a href="#orgc2ca77d">2.2. Folding Simulation Methods</a>
<ul>
<li><a href="#org83bb493">2.2.1. all-atom molecular dynamics (MD)</a></li>
<li><a href="#org22d70e9">2.2.2. Accelerated molecular dynamics (AMD)</a></li>
</ul>
</li>
<li><a href="#orgfb09aa8">2.3. Voltage gated ion channels</a>
<ul>
<li><a href="#org591d4a0">2.3.1. overview</a></li>
<li><a href="#org4dd4606">2.3.2. mechanical function</a></li>
<li><a href="#org71039e6">2.3.3. related components</a></li>
<li><a href="#org8f24dbb">2.3.4. sources&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#org7766f3d">3. Meetings</a>
<ul>
<li><a href="#org81a21dd">3.1. 12 oct 2020</a>
<ul>
<li><a href="#org25c6291">3.1.1. protien synthase</a></li>
<li><a href="#org320c10f">3.1.2. neurotransmitters</a></li>
<li><a href="#orga104957">3.1.3. Voltage Driven Things</a></li>
<li><a href="#org38a1d7e">3.1.4. Case study</a></li>
<li><a href="#org1daf0e7">3.1.5. prions</a></li>
</ul>
</li>
<li><a href="#org936f9ff">3.2. 29 Oct 2020</a></li>
<li><a href="#org315c32b">3.3. 03 nov 2020</a>
<ul>
<li><a href="#org21a7294">3.3.1. apparently I can just turn in this notes document, and the detail is good enough</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#org20e3e96">4. Poster Plannning</a>
<ul>
<li><a href="#org3f4ee9f">4.1. nerve cell, action potential, depolarization</a>
<ul>
<li><a href="#orgc575d4b">4.1.1. resources</a></li>
<li><a href="#org344e54f">4.1.2. membrane potential graph</a></li>
</ul>
</li>
<li><a href="#org628becb">4.2. structure of VGIC + conformational change within subunit</a>
<ul>
<li><a href="#orgd2f5bf9">4.2.1. resources&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span></a></li>
</ul>
</li>
<li><a href="#orga5c3cba">4.3. electrostatic interactions of S4</a>
<ul>
<li><a href="#orgd1c2686">4.3.1. resources&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span></a></li>
</ul>
</li>
<li><a href="#orgac2846c">4.4. bottom diagrams</a>
<ul>
<li><a href="#org8298653">4.4.1. Nav</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</div></div>
</div>

<div id="outline-container-org00b9a85" class="outline-2">
<h2 id="org00b9a85"><span class="section-number-2">1</span> Preliminary Research</h2>
<div class="outline-text-2" id="text-1">
</div>
<div id="outline-container-org7e13db1" class="outline-3">
<h3 id="org7e13db1"><span class="section-number-3">1.1</span> Sources</h3>
<div class="outline-text-3" id="text-1-1">
<p>
<a href="https://www.frontiersin.org/articles/10.3389/fchem.2019.00540/ful">https://www.frontiersin.org/articles/10.3389/fchem.2019.00540/ful</a>
</p>
</div>
</div>
</div>

<div id="outline-container-org04077bc" class="outline-2">
<h2 id="org04077bc"><span class="section-number-2">2</span> Notes</h2>
<div class="outline-text-2" id="text-2">
</div>
<div id="outline-container-org5fd3bb5" class="outline-3">
<h3 id="org5fd3bb5"><span class="section-number-3">2.1</span> Target Processes</h3>
<div class="outline-text-3" id="text-2-1">
</div>
<div id="outline-container-org555952b" class="outline-4">
<h4 id="org555952b"><span class="section-number-4">2.1.1</span> Enzyme catalysis</h4>
<div class="outline-text-4" id="text-2-1-1">
<p>
catalyzing reactions with actions
</p>
</div>
</div>

<div id="outline-container-orge669a25" class="outline-4">
<h4 id="orge669a25"><span class="section-number-4">2.1.2</span> Protein-ligand binding</h4>
<div class="outline-text-4" id="text-2-1-2">
<p>
neurotransmitters (dopamine), protien is dopamine receptor
how does the ligand bind the proper site to open the channel?
</p>
<ul class="org-ul">
<li>ligand: how to pronounce?</li>
</ul>
</div>
</div>

<div id="outline-container-org29b1964" class="outline-4">
<h4 id="org29b1964"><span class="section-number-4">2.1.3</span> signal transduction</h4>
<div class="outline-text-4" id="text-2-1-3">
<p>
bind to other protein to trigger chain of actions
</p>
<ul class="org-ul">
<li>release calcium from intercellular stores</li>
</ul>
</div>
</div>

<div id="outline-container-orgdd3d8aa" class="outline-4">
<h4 id="orgdd3d8aa"><span class="section-number-4">2.1.4</span> allosteric regulation</h4>
<div class="outline-text-4" id="text-2-1-4">
<ul class="org-ul">
<li>a reason why knowing the structure and pockets is important</li>
<li>predict allosteric cites
<ul class="org-ul">
<li>similar to non-competative inhibiton?</li>
<li>but for dna binding protiens, like the dna transcription inhibitor</li>
</ul></li>
<li>ligand binds allosteric site and activates the protien</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgc2ca77d" class="outline-3">
<h3 id="orgc2ca77d"><span class="section-number-3">2.2</span> Folding Simulation Methods</h3>
<div class="outline-text-3" id="text-2-2">
</div>
<div id="outline-container-org83bb493" class="outline-4">
<h4 id="org83bb493"><span class="section-number-4">2.2.1</span> all-atom molecular dynamics (MD)</h4>
<div class="outline-text-4" id="text-2-2-1">
<ul class="org-ul">
<li>Obtains all desired information regarding the kinetics and thermodynamics</li>
</ul>
</div>

<ol class="org-ol">
<li><a id="org7a69c06"></a>Time scale bottleneck<br />
<div class="outline-text-5" id="text-2-2-1-1">
<ul class="org-ul">
<li>very slow (supercomputers -&gt; microseconds of simulation)</li>
<li>require microsecond to milisecond time scales</li>
</ul>
</div>

<ol class="org-ol">
<li><a id="org2c79008"></a>optimizations<br />
<ol class="org-ol">
<li><a id="org1985559"></a>conformational sampling?<br />
<div class="outline-text-7" id="text-2-2-1-1-1-1">
<ul class="org-ul">
<li>retains atomistic representation of the system</li>
</ul>
</div>
</li>

<li><a id="org528cf81"></a>overcome kinetic trapping and thourough sampling of conformational space techniques<br />
<div class="outline-text-7" id="text-2-2-1-1-1-2">
<ul class="org-ul">
<li>umbrella sampling</li>
<li>multicanonical algorithms</li>
<li>simulated tempering</li>
<li>transition path sampling</li>
<li>targeted molecular dynamics</li>
<li>replica exchange method molecular dynamics (REMD)</li>
<li>accelerated molecular dynamics (AMD)
<ul class="org-ul">
<li>see below</li>
</ul></li>
</ul>
</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
</div>

<div id="outline-container-org22d70e9" class="outline-4">
<h4 id="org22d70e9"><span class="section-number-4">2.2.2</span> Accelerated molecular dynamics (AMD)</h4>
<div class="outline-text-4" id="text-2-2-2">
<p>
epic
</p>
</div>
</div>
</div>

<div id="outline-container-orgfb09aa8" class="outline-3">
<h3 id="orgfb09aa8"><span class="section-number-3">2.3</span> Voltage gated ion channels</h3>
<div class="outline-text-3" id="text-2-3">
</div>
<div id="outline-container-org591d4a0" class="outline-4">
<h4 id="org591d4a0"><span class="section-number-4">2.3.1</span> overview</h4>
<div class="outline-text-4" id="text-2-3-1">
</div>
<ol class="org-ol">
<li><a id="org73819f9"></a>lives on cell membrane<br /></li>
<li><a id="org2e49d76"></a>role<br />
<ol class="org-ol">
<li><a id="org5e3054b"></a>allows ions in/out<br /></li>
<li><a id="org6750131"></a>crucial in "excitable" cells, like neurons<br /></li>
<li><a id="org7cc2e81"></a>propogates elecrical signals directionally<br /></li>
<li><a id="org8679c84"></a>ion specific<br />
<ol class="org-ol">
<li><a id="orgd41417d"></a>Na^+<br /></li>
<li><a id="org0e9f382"></a>K^+<br /></li>
<li><a id="org33e7432"></a>Ca<sup>2+</sup><br /></li>
<li><a id="org5979eed"></a>Cl^-<br /></li>
</ol>
</li>
<li><a id="org5905d4b"></a>triggered by voltage across cell membrane<br /></li>
</ol>
</li>
<li><a id="orgc193e13"></a>parts<br />
<ol class="org-ol">
<li><a id="org19578ea"></a>voltage sensor<br /></li>
<li><a id="orge957aae"></a>pore/conducting pathway<br /></li>
<li><a id="org262acb2"></a>gate<br /></li>
</ol>
</li>
<li><a id="org630ce72"></a><b>sodium/calcium channels</b><br />
<ol class="org-ol">
<li><a id="orgbd98f54"></a>parts<br />
<ol class="org-ol">
<li><a id="orga592d6a"></a>one polypeptide<br /></li>
<li><a id="org32bcb10"></a>creates "four homologous domains"<br />
<ol class="org-ol">
<li><a id="org52b3c98"></a>each one consists of 6 membrane spanning alpha helices<br /></li>
</ol>
</li>
<li><a id="orgbc6698e"></a>the S4 helix [of each domain?] is the voltage sensing helix<br />
<ol class="org-ol">
<li><a id="orgc9b1038"></a>This helix contains enough positive charges to feel an electrostatic repelling force from the high charge outside the cell.<br />
<ol class="org-ol">
<li><a id="orgd979855"></a>lysine or arginine "residues in repeated motifs"<br /></li>
<li><a id="orgeebaa93"></a>in resting state, half of each S4 helix is in contact with the cell cytosol<br /></li>
<li><a id="org4acba5c"></a>upon depolarization, positive residues move towards surface of membrane?<br /></li>
<li><a id="orgaee012a"></a>movement triggers comformational change in the gate<br /></li>
</ol>
</li>
</ol>
</li>
<li><a id="org3e83db0"></a>s6 domain<br />
<ol class="org-ol">
<li><a id="org47169a9"></a>thought to mechanically block the ions from passing through the channel<br /></li>
</ol>
</li>
<li><a id="org62abe17"></a>inactivation gate<br />
<ol class="org-ol">
<li><a id="org72e173b"></a>structure<br />
<div class="outline-text-8" id="text-2-3-1-4-1-5-1">
<p>
Another gate that stops ions from flowing, [giving the main gate more to reset?]
</p>
</div>
<ol class="org-ol">
<li><a id="orgd2db304"></a>modeled as a ball tethered to a flexible chain<br /></li>
<li><a id="orga010d5a"></a>the chain is supposed to fold up on itself to pull the ball in and block ion flow<br /></li>
</ol>
</li>
</ol>
</li>
</ol>
</li>
</ol>
</li>
</ol>
</div>

<div id="outline-container-org4dd4606" class="outline-4">
<h4 id="org4dd4606"><span class="section-number-4">2.3.2</span> mechanical function</h4>
<div class="outline-text-4" id="text-2-3-2">
</div>
<ol class="org-ol">
<li><a id="org8dbe51d"></a>sources<br />
<ol class="org-ol">
<li><a id="orge89c335"></a><a href="https://www.pnas.org/content/112/1/124">https://www.pnas.org/content/112/1/124</a>&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span><br /></li>

<li><a id="orga917672"></a><a href="https://www.sciencedirect.com/science/article/pii/S0076687918300156">https://www.sciencedirect.com/science/article/pii/S0076687918300156</a>&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span><br /></li>
</ol>
</li>

<li><a id="org991f8e4"></a>explanation<br />
<ol class="org-ol">
<li><a id="orge634592"></a>structural overview<br />
<div class="outline-text-6" id="text-2-3-2-2-1">
<p>
In each analogous subdomain, the segment S4 (of 6) (where the first four are voltage sensing, S5, S6 form the pore, and the S4-S5 linker is important but ellusive) is quite positively charged (3-7 positive Rgroups like Agrinine?)
</p>
</div>
</li>

<li><a id="org6c076c8"></a>salt bridge pattern rearranges?<br /></li>

<li><a id="orgf287c86"></a>something about gating currents and inactivation<br /></li>
</ol>
</li>
</ol>
</div>

<div id="outline-container-org71039e6" class="outline-4">
<h4 id="org71039e6"><span class="section-number-4">2.3.3</span> related components</h4>
<div class="outline-text-4" id="text-2-3-3">
</div>
<ol class="org-ol">
<li><a id="orgfd3ec66"></a>membrane depolarization<br />
<div class="outline-text-5" id="text-2-3-3-1">
<p>
The interior of the cell temporarily becomes more positive (less negative) than the exterior
</p>
</div>
<ol class="org-ol">
<li><a id="org0e2391d"></a><a href="https://en.wikipedia.org/wiki/Depolarization">https://en.wikipedia.org/wiki/Depolarization</a>&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span><br /></li>
</ol>
</li>
<li><a id="orga069ceb"></a>membrane potential<br />
<div class="outline-text-5" id="text-2-3-3-2">
<p>
the "default" charge/voltage difference accross a cell membrane
</p>
<ul class="org-ul">
<li>the inside is usually more negative</li>
</ul>
</div>
<ol class="org-ol">
<li><a id="orgf18a7fd"></a><a href="https://en.wikipedia.org/wiki/Ball_and_chain_inactivation">https://en.wikipedia.org/wiki/Ball_and_chain_inactivation</a>&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span><br /></li>
</ol>
</li>
<li><a id="orgb1a9071"></a>graded potentials<br />
<ol class="org-ol">
<li><a id="orge5e82d8"></a>a "smallish change in the membrane potential that is porportional to the size of the stimulus"<br />
<ol class="org-ol">
<li><a id="org34b8029"></a>doesn't travel a long distance<br /></li>
<li><a id="orgda79126"></a>diminishes/fades away as its spreads<br /></li>
</ol>
</li>
<li><a id="org8ce4e5e"></a><a href="https://www.khanacademy.org/science/biology/human-biology/neuron-nervous-system/a/depolarization-hyperpolarization-and-action-potentials">https://www.khanacademy.org/science/biology/human-biology/neuron-nervous-system/a/depolarization-hyperpolarization-and-action-potentials</a>&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span><br /></li>
</ol>
</li>
<li><a id="org4a79862"></a>action potential<br />
<ol class="org-ol">
<li><a id="orgfe0ffcc"></a>always the same size<br /></li>
<li><a id="org8c81c57"></a>binary (all or none)<br /></li>
<li><a id="org04337c1"></a>happens when depolarization increases the membrane voltage across a threshold value (usually about -55mV)<br /></li>
<li><a id="org3db130a"></a>causes voltage gated Na^+ channels to open<br /></li>
<li><a id="org80f79c5"></a>voltage goes up quickl to around 40mV (positive)<br /></li>
<li><a id="org742b554"></a>after some time, Na+ VGICs inactivate<br /></li>
<li><a id="org3e0eb91"></a>potassium channels stay open a little longer to bring the membrane potential back<br /></li>
<li><a id="orgde82d7a"></a>sodium channels return to normal state (still closed, but can respond to voltage again)<br /></li>
<li><a id="org8898a08"></a>"refactor period ensures that the action potential will only travel forward down the axon, not backwards through the portion of the axon that just underwent an action potential"<br /></li>
</ol>
</li>
<li><a id="org6ee0e19"></a>Impulse speed<br />
<ol class="org-ol">
<li><a id="org8325396"></a>larger diameter axon<br />
<div class="outline-text-6" id="text-2-3-3-5-1">
<p>
A greater diameter will allow the action potential to travel faster because there are structures in the cytoplasm of each axon to block the ions' travel. However, with a larger diameter, there are more paths for the ion to travel though, even if the concentration is the same? (because there is more volume to surface area? and a direction is a point on surface area but the ion is a point in volume?)
</p>
</div>
</li>
<li><a id="org12b6624"></a>Mylelin sheath<br />
<div class="outline-text-6" id="text-2-3-3-5-2">
<p>
Increases the distances between cations and anions on opposite sides of the axon membrane, which decreases capacitence (yes physics capacitence). So less charge can be tied to the membrane, so depolarization happens faster (fewer charges need to move).
</p>
</div>
<ol class="org-ol">
<li><a id="org1fd376f"></a><a href="https://www.khanacademy.org/science/health-and-medicine/nervous-system-and-sensory-infor/neuron-membrane-potentials-topic/v/effects-of-axon-diameter-and-myelination">https://www.khanacademy.org/science/health-and-medicine/nervous-system-and-sensory-infor/neuron-membrane-potentials-topic/v/effects-of-axon-diameter-and-myelination</a>&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span><br /></li>
</ol>
</li>
</ol>
</li>
<li><a id="orgff39ee3"></a>Salt bridge<br />
<div class="outline-text-5" id="text-2-3-3-6">
<p>
When two oppositely charged R-groups are close enough together to experience electrostatic attraction
</p>
</div>
</li>
</ol>
</div>

<div id="outline-container-org8f24dbb" class="outline-4">
<h4 id="org8f24dbb"><span class="section-number-4">2.3.4</span> sources&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span></h4>
<div class="outline-text-4" id="text-2-3-4">
<ul class="org-ul">
<li><a href="https://en.wikipedia.org/wiki/Voltage-gated_ion_channel">https://en.wikipedia.org/wiki/Voltage-gated_ion_channel</a></li>
<li><a href="https://www.sciencedirect.com/science/article/pii/S0076687918300156">Computational Approaches to Studying Voltage-Gated Ion Channel Modulation by General Anesthetics - ScienceDirect</a></li>
<li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S0092867419307342-fx1_lrg.jpg">1-s2.0-S0092867419307342-fx1<sub>lrg.jpg</sub> (996×996)</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3266868/">THE CRYSTAL STRUCTURE OF A VOLTAGE-GATED SODIUM CHANNEL</a></li>
<li><a href="https://www.pnas.org/content/112/1/124/tab-figures-data">Free-energy landscape of ion-channel voltage-sensor–domain activation | PNAS</a>
<ul class="org-ul">
<li><a href="https://www.pnas.org/content/pnas/112/1/124/F1.large.jpg">F1.large.jpg (1036×1280)</a></li>
</ul></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2950829/">Ion Channel Voltage Sensors: Structure, Function, and Pathophysiology</a></li>
<li><a href="https://www.sciencedirect.com/science/article/abs/pii/S0959440X16301506">Voltage-gated sodium channels viewed through a structural biology lens - ScienceDirect</a></li>
<li><a href="https://www.khanacademy.org/science/health-and-medicine/nervous-system-and-sensory-infor/neuron-membrane-potentials-topic/v/effects-of-axon-diameter-and-myelination">Effects of axon diameter and myelination (video) | Khan Academy</a></li>
<li><a href="https://www.sciencedirect.com/science/article/pii/S0006349514007875">Probing α-310 Transitions in a Voltage-Sensing S4 Helix - ScienceDirect</a></li>
<li><a href="https://www.cell.com/structure/fulltext/S0969-2126(15)00500-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0969212615005006%3Fshowall%3Dtrue">Molecular Interactions in the Voltage Sensor Controlling Gating Properties of CaV Calcium Channels: Structure</a></li>
<li><a href="https://www.sciencedirect.com/science/article/pii/S0092867419307342">Resting-State Structure and Gating Mechanism of a Voltage-Gated Sodium Channel - ScienceDirect</a></li>
<li><a href="https://www.pnas.org/content/109/2/E93">Structural basis for gating charge movement in the voltage sensor of a sodium channel | PNAS</a></li>
<li><a href="https://science.sciencemag.org/content/362/6412/eaau2596?rss%253D1=">Structural basis for the modulation of voltage-gated sodium channels by animal toxins | Science</a></li>
<li><a href="https://pharmrev.aspetjournals.org/content/57/4/387.full">Overview of Molecular Relationships in the Voltage-Gated Ion Channel Superfamily | Pharmacological Reviews</a></li>
<li><a href="https://physoc.onlinelibrary.wiley.com/doi/10.1113/expphysiol.2013.071969">Structure and function of voltage‐gated sodium channels at atomic resolution - Catterall - 2014 - Experimental Physiology - Wiley Online Library</a></li>
<li><a href="https://www.cell.com/cell/fulltext/S0092-8674(11)00997-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867411009974%3Fshowall%3Dtrue">Crystal Structure of the Mammalian GIRK2 K+ Channel and Gating Regulation by G Proteins, PIP2, and Sodium: Cell</a></li>
<li><a href="https://www.sciencedirect.com/science/article/pii/S2211124719301512">The Role of CaV2.1 Channel Facilitation in Synaptic Facilitation - ScienceDirect</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3448954/">The sliding-helix voltage sensor</a></li>
<li><a href="https://www.pnas.org/content/103/19/7292">Voltage sensor conformations in the open and closed states in rosetta structural models of K+ channels | PNAS</a></li>
<li><a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/voltage-gated-ion-channel#:~:text=Voltage%2Dgated%20ion%20channels%20contain,domain%20responsible%20for%20sensing%20voltage.">Voltage-Gated Ion Channel - an overview | ScienceDirect Topics</a></li>
</ul>
</div>
</div>
</div>
</div>

<div id="outline-container-org7766f3d" class="outline-2">
<h2 id="org7766f3d"><span class="section-number-2">3</span> Meetings</h2>
<div class="outline-text-2" id="text-3">
</div>
<div id="outline-container-org81a21dd" class="outline-3">
<h3 id="org81a21dd"><span class="section-number-3">3.1</span> 12 oct 2020</h3>
<div class="outline-text-3" id="text-3-1">
<ul class="org-ul">
<li>computational prediction modeling
<ul class="org-ul">
<li>trying to predict the crystal structure
<ul class="org-ul">
<li>why?
<ul class="org-ul">
<li>to analyze would this fit?</li>
<li>does it work with this target</li>
</ul></li>
</ul></li>
</ul></li>
<li>solving the structure
<ul class="org-ul">
<li>xray cristolography
<ul class="org-ul">
<li>gold standard</li>
<li>now got the structure
<ul class="org-ul">
<li>what does that mean?</li>
<li>can we simulate how it interacts?</li>
<li>can you then do modeling on that to see if drug molecules work? are useful</li>
</ul></li>
</ul></li>
</ul></li>
<li>look at some concrete examples?</li>
<li>tell a biological story alongside with computational relevance piece</li>
</ul>
</div>

<div id="outline-container-org25c6291" class="outline-4">
<h4 id="org25c6291"><span class="section-number-4">3.1.1</span> protien synthase</h4>
<div class="outline-text-4" id="text-3-1-1">
<p>
not as much simulation stuff
</p>
</div>
</div>

<div id="outline-container-org320c10f" class="outline-4">
<h4 id="org320c10f"><span class="section-number-4">3.1.2</span> neurotransmitters</h4>
<div class="outline-text-4" id="text-3-1-2">
<p>
dopamine
sodium rushes in, electrochemical and concentration gradient
recharge gradient by releasing potassium
</p>
</div>

<ol class="org-ol">
<li><a id="org1f42310"></a>nerst equation<br />
<div class="outline-text-5" id="text-3-1-2-1">
<p>
electrochemical gradient as battery
</p>
</div>
</li>

<li><a id="org43b2929"></a>goldman-katz equation<br />
<div class="outline-text-5" id="text-3-1-2-2">
<ul class="org-ul">
<li>applied to neuro</li>
<li>takes into account the concentrations of the 4 ions
<ul class="org-ul">
<li>how does the power of the battery work given those components?</li>
<li>ligands and pH can change/denature protiens, but there are also voltage gated channels</li>
</ul></li>
</ul>
</div>
</li>
</ol>
</div>

<div id="outline-container-orga104957" class="outline-4">
<h4 id="orga104957"><span class="section-number-4">3.1.3</span> Voltage Driven Things</h4>
<div class="outline-text-4" id="text-3-1-3">
<ul class="org-ul">
<li>Heartbeart</li>
<li>nervous system
<ul class="org-ul">
<li>how do voltage gated ion channels work?</li>
</ul></li>
</ul>
</div>

<ol class="org-ol">
<li><a id="org8af32c6"></a>things to know about<br />
<div class="outline-text-5" id="text-3-1-3-1">
<ul class="org-ul">
<li>action potential</li>
<li>voltage gated calcium channels open at depolarization threshold</li>
</ul>
</div>

<ol class="org-ol">
<li><a id="orgd88cc70"></a>neurotransmitters<br />
<div class="outline-text-6" id="text-3-1-3-1-1">
<ul class="org-ul">
<li>"calcium mediated exocitosis of neurotransmitter vesicles in the synaptic terminal"</li>
<li>calcium rushes somewhere to allow the neurotransmitters to leave the cell</li>
</ul>
</div>
</li>
</ol>
</li>
</ol>
</div>

<div id="outline-container-org38a1d7e" class="outline-4">
<h4 id="org38a1d7e"><span class="section-number-4">3.1.4</span> Case study</h4>
<div class="outline-text-4" id="text-3-1-4">
<ul class="org-ul">
<li>why do we care? why is this useful</li>
<li>knowing the structure can lead to some useful information</li>
<li>how did it lead to some sort of accelerated understanding?</li>
</ul>
</div>
</div>

<div id="outline-container-org1daf0e7" class="outline-4">
<h4 id="org1daf0e7"><span class="section-number-4">3.1.5</span> prions</h4>
<div class="outline-text-4" id="text-3-1-5">
<ul class="org-ul">
<li>how to pronounce?</li>
</ul>
</div>

<ol class="org-ol">
<li><a id="org7ac6044"></a>CJD<br />
<div class="outline-text-5" id="text-3-1-5-1">
<ul class="org-ul">
<li>is it inheritable?</li>
<li>one case per million population</li>
</ul>
</div>

<ol class="org-ol">
<li><a id="orgf9c0541"></a>Casues<br />
<div class="outline-text-6" id="text-3-1-5-1-1">
<ul class="org-ul">
<li>the gene that causes CJD in 5-10% of cases is PRNP</li>
<li>87% of cases are sporatic</li>
</ul>
</div>
</li>
</ol>
</li>

<li><a id="orga8e6367"></a>isoform<br />
<div class="outline-text-5" id="text-3-1-5-2">
<ul class="org-ul">
<li>a different set of intons and exons</li>
<li>splicosome takes pre-RNA and cuts out intons
<ul class="org-ul">
<li>even if the pre-RNA had 10 exons, the splicosome might take a subset of those exons and remove the others</li>
</ul></li>
<li>An isoform is a variant of that subset, an abnormal isoform is one that is "bad" and causes problems</li>
</ul>
</div>
</li>
</ol>
</div>
</div>

<div id="outline-container-org936f9ff" class="outline-3">
<h3 id="org936f9ff"><span class="section-number-3">3.2</span> 29 Oct 2020</h3>
<div class="outline-text-3" id="text-3-2">
<p>
uh nothing happened
</p>
</div>
</div>

<div id="outline-container-org315c32b" class="outline-3">
<h3 id="org315c32b"><span class="section-number-3">3.3</span> 03 nov 2020</h3>
<div class="outline-text-3" id="text-3-3">
</div>
<div id="outline-container-org21a7294" class="outline-4">
<h4 id="org21a7294"><span class="section-number-4">3.3.1</span> apparently I can just turn in this notes document, and the detail is good enough</h4>
</div>
</div>
</div>

<div id="outline-container-org20e3e96" class="outline-2">
<h2 id="org20e3e96"><span class="section-number-2">4</span> Poster Plannning</h2>
<div class="outline-text-2" id="text-4">
</div>
<div id="outline-container-org3f4ee9f" class="outline-3">
<h3 id="org3f4ee9f"><span class="section-number-3">4.1</span> nerve cell, action potential, depolarization</h3>
<div class="outline-text-3" id="text-4-1">
</div>
<div id="outline-container-orgc575d4b" class="outline-4">
<h4 id="orgc575d4b"><span class="section-number-4">4.1.1</span> resources</h4>
<div class="outline-text-4" id="text-4-1-1">
<ul class="org-ul">
<li><a href="https://www.khanacademy.org/science/biology/human-biology/neuron-nervous-system/a/depolarization-hyperpolarization-and-action-potentials">Khan Academy on Depolarization, hyperpolarization, and action potentials + some epic comments</a></li>
</ul>
</div>
</div>

<div id="outline-container-org344e54f" class="outline-4">
<h4 id="org344e54f"><span class="section-number-4">4.1.2</span> membrane potential graph</h4>
</div>
</div>

<div id="outline-container-org628becb" class="outline-3">
<h3 id="org628becb"><span class="section-number-3">4.2</span> structure of VGIC + conformational change within subunit</h3>
<div class="outline-text-3" id="text-4-2">
</div>
<div id="outline-container-orgd2f5bf9" class="outline-4">
<h4 id="orgd2f5bf9"><span class="section-number-4">4.2.1</span> resources&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span></h4>
<div class="outline-text-4" id="text-4-2-1">
<ul class="org-ul">
<li><a href="https://www.sciencedirect.com/science/article/pii/S0076687918300156">Computational Approaches to Studying Voltage-Gated Ion Channel Modulation by General Anesthetics - ScienceDirect</a></li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orga5c3cba" class="outline-3">
<h3 id="orga5c3cba"><span class="section-number-3">4.3</span> electrostatic interactions of S4</h3>
<div class="outline-text-3" id="text-4-3">
</div>
<div id="outline-container-orgd1c2686" class="outline-4">
<h4 id="orgd1c2686"><span class="section-number-4">4.3.1</span> resources&#xa0;&#xa0;&#xa0;<span class="tag"><span class="source">source</span></span></h4>
<div class="outline-text-4" id="text-4-3-1">
<ul class="org-ul">
<li><a href="https://www.pnas.org/content/112/1/124/tab-figures-data">Free-energy landscape of ion-channel voltage-sensor–domain activation | PNAS</a>
<ul class="org-ul">
<li><a href="https://www.pnas.org/content/pnas/112/1/124/F1.large.jpg">F1.large.jpg (1036×1280)</a></li>
</ul></li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgac2846c" class="outline-3">
<h3 id="orgac2846c"><span class="section-number-3">4.4</span> bottom diagrams</h3>
<div class="outline-text-3" id="text-4-4">
</div>
<div id="outline-container-org8298653" class="outline-4">
<h4 id="org8298653"><span class="section-number-4">4.4.1</span> Nav</h4>
<div class="outline-text-4" id="text-4-4-1">
</div>
<ol class="org-ol">
<li><a id="orga17ef88"></a>side view, all alpha domains<br /></li>

<li><a id="orgb8b6280"></a>top view, all alpha domains<br />
<ol class="org-ol">
<li><a id="orgc0a221c"></a>open/close?<br />
<div class="outline-text-6" id="text-4-4-1-2-1">
</div>
</li>
</ol>
</li>
</ol>
</div>
</div>
</div>
</div>
</body>
</html>
