% Created 2021-09-12 Sun 22:49
% Intended LaTeX compiler: xelatex
\documentclass[letterpaper]{article}
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{longtable}
\usepackage{wrapfig}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{amsmath}
\usepackage{textcomp}
\usepackage{amssymb}
\usepackage{capt-of}
\usepackage{hyperref}
\usepackage[margin=1in]{geometry}
\usepackage{fontspec}
\usepackage{indentfirst}
\setmainfont[ItalicFont = LiberationSans-Italic, BoldFont = LiberationSans-Bold, BoldItalicFont = LiberationSans-BoldItalic]{LiberationSans}
\newfontfamily\NHLight[ItalicFont = LiberationSansNarrow-Italic, BoldFont       = LiberationSansNarrow-Bold, BoldItalicFont = LiberationSansNarrow-BoldItalic]{LiberationSansNarrow}
\newcommand\textrmlf[1]{{\NHLight#1}}
\newcommand\textitlf[1]{{\NHLight\itshape#1}}
\let\textbflf\textrm
\newcommand\textulf[1]{{\NHLight\bfseries#1}}
\newcommand\textuitlf[1]{{\NHLight\bfseries\itshape#1}}
\usepackage{fancyhdr}
\pagestyle{fancy}
\usepackage{titlesec}
\usepackage{titling}
\makeatletter
\lhead{\textbf{\@title}}
\makeatother
\rhead{\textrmlf{Compiled} \today}
\lfoot{\theauthor\ \textbullet \ \textbf{2021-2022}}
\cfoot{}
\rfoot{\textrmlf{Page} \thepage}
\titleformat{\section} {\Large} {\textrmlf{\thesection} {|}} {0.3em} {\textbf}
\titleformat{\subsection} {\large} {\textrmlf{\thesubsection} {|}} {0.2em} {\textbf}
\titleformat{\subsubsection} {\large} {\textrmlf{\thesubsubsection} {|}} {0.1em} {\textbf}
\setlength{\parskip}{0.45em}
\renewcommand\maketitle{}
\author{Houjun Liu}
\date{\today}
\title{Single Nucleotide Polymorphism}
\hypersetup{
 pdfauthor={Houjun Liu},
 pdftitle={Single Nucleotide Polymorphism},
 pdfkeywords={},
 pdfsubject={},
 pdfcreator={Emacs 28.0.50 (Org mode 9.4.4)}, 
 pdflang={English}}
\begin{document}

\maketitle


\section{SNP}
\label{sec:org234c01d}
\textbf{Single Nucleotipe Polymorphism}: pairs of single-base pair variations
of genes that are not uncommon (>1\%), and hence not quite considered a
mutation.

SNPs could be in any part of the genome: sometimes in a gene, but more
often in a non-coding region.

\begin{figure}[htbp]
\centering
\includegraphics[width=.9\linewidth]{./Pasted image 20210505134813.png}
\caption{Pasted image 20210505134813.png}
\end{figure}

These could do absolutely nothing, but they could also create large
trait differences (sickle-cell is a SNP).

\emph{The DNA by mail kits are actually just looking for SNPs!}

Multiple SNPs come together to form a "Haplotype"; they are SNPs that
are fairly close together that frequently occur in sequence (like the
chain of three SNPs A something something G something something T would
be a haplotype.)

People of the same haplotype are likely of same ancestry.

\subsection{SNP Guidelines}
\label{sec:org0b34b90}
\begin{itemize}
\item No medical SNPs; so avoid SNPs that could be associated with heavy
medical information
\item No SNPs that could potentially give you PTSD
\end{itemize}

\subsection{the Reaction}
\label{sec:org0c012d1}
\begin{itemize}
\item Each student's PCR reaction will only amplify 500-800 nucleotide
region
\item Find SNPs that have specific traits independently, because most SNPs
are polygenic (they tell you info only if you discover many different
SNPs, which is complicated and hard)
\item SNPs are sometimes discovered using Genome Wide Association Studies,
which is challenging to deal with because they may be flawed
\end{itemize}

\noindent\rule{\textwidth}{0.5pt}

\subsection{Polymerase Chain Reaction (PCR)}
\label{sec:orgc77bde1}
\emph{A reaction by which the DNA replication process is attempted to be
replicated out of the cell in preparation to sequence one's DNA}

\textbf{PCR provides a way of creating \emph{lots} of copies of a \emph{small} strand of
DNA}

\subsubsection{Reaction Need}
\label{sec:org47b2ac4}
\begin{itemize}
\item The DNA Portion (duh)
\item A buffer to host the liquid
\item Primers - Bit of RNA to help DNA polymerase get going
\item DNA Polymerase - The Enzyme to Copy the DNA => Typically, the TAQ
Polymerease is used because it is from a bacterial that's quite
resistant to heat
\item DNA Nucleotides - to build the replicated DNA
\end{itemize}

\subsubsection{Steps}
\label{sec:orgd7fe9cc}
\begin{enumerate}
\item Denaturation

\begin{enumerate}
\item Cook the DNA ("denature it"/heat it) to seperate the two strands
of the DNA
\end{enumerate}

\item Annealing

\begin{enumerate}
\item Cool the Cooked DNA, and dump in the primers
\item The specific temperature in this step is important because it
controlls where the primer binds, and hence which part of the DNA
you want to amplify
\end{enumerate}

\item DNA Synthesis => make more copies of the DNA

\begin{enumerate}
\item This needs a slightly higher temperature than annealing, but which
one depends on what polymerase you used
\end{enumerate}

\item Go to step 1
\end{enumerate}

Note: the DNA polymerase may run over, so we just keep repeating this
and it will slowly become more successful as the DNA polymerease repeats
its work repeatedly.

BTW: if you read carefully, you will realize that you made TWO DNA
double-strands out of a single double strand. So exponential growth
baybee and then you could get lots of copies of that region.

\subsubsection{But, why?}
\label{sec:org0f134ce}
In order to have enough copies for gel electroprorisis (like, the
rRT-PCR COVID test) or, for things like crime forensics, you need many
copies of the same DNA for a clearly detectable result for analysis that
may have a high threshold.
\end{document}
