<html><head><meta charset="utf-8" /><meta content="width=device-width, initial-scale=1.0" name="viewport" /><meta content="Taproot Authors" name="author" /><meta content="A collection of academic and project notes" name="description" /><title>Taproot</title><link href="/static/css/firn_base.css" rel="stylesheet" /><link href="/static/css/prism.css" rel="stylesheet" /><link href="/static/css/global.css" rel="stylesheet" /><script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_HTML" type="text/javascript"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/prism.min.js" type="text/javascript"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
        'displayAlign': 'center',
        'displayIndent': '0em',
        'extensions': ['tex2jax.js'],
        'tex2jax': {
        'inlineMath': [ ['$','$'], ['\\(','\\)'] ],
        'processEscapes': true
        },
        'HTML-CSS': { scale: 100,
                        linebreaks: { automatic: 'false' },
                        webFont: 'TeX'
                       },
        SVG: {scale: 100,
              linebreaks: { automatic: 'false' },
              font: 'TeX'},
        NativeMML: {scale: 100},
        TeX: { equationNumbers: {autoNumber: 'AMS'},
               MultLineWidth: '85%',
               TagSide: 'right',
               TagIndent: '.8em',
tex: {
autoload: {
                cases: [[], ['numcases', 'subnumcases']]
        }
}}})</script></head><body><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/components/prism-core.min.js" type="text/javascript"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/plugins/autoloader/prism-autoloader.min.js" type="text/javascript"></script><div class="headerline"><a class="wordmark" href="https://taproot3.sanity.gq/" style="border:0">TR3</a></div><main><article class="content rss"><div class="preamble"><h1 class="title">Single Nucleotide Polymorphism</h1></div><div class="metamble"><span><span>Written by </span><span>Houjun Liu</span></span></div><div class="notebody"><div><section></section><div class="firn-headline-section firn-headline-section-1"><h1 class="firn-headline firn-headline-1" id="snp"><span class="firn-headline-text"><span>SNP</span></span></h1><section><p><strong><span>Single Nucleotipe Polymorphism</span></strong><span>: pairs of single-base pair variations
of genes that are not uncommon (>1%), and hence not quite considered a
mutation.</span></p><p><span>SNPs could be in any part of the genome: sometimes in a gene, but more
often in a non-coding region.</span></p><p><a href="#pasted-image-20210505134813png"></a></p><p><span>These could do absolutely nothing, but they could also create large
trait differences (sickle-cell is a SNP).</span></p><p><em><span>The DNA by mail kits are actually just looking for SNPs!</span></em></p><p><span>Multiple SNPs come together to form a "Haplotype"; they are SNPs that
are fairly close together that frequently occur in sequence (like the
chain of three SNPs A something something G something something T would
be a haplotype.)</span></p><p><span>People of the same haplotype are likely of same ancestry.</span></p></section><div class="firn-headline-section firn-headline-section-2"><h2 class="firn-headline firn-headline-2" id="snp-guidelines"><span class="firn-headline-text"><span>SNP Guidelines</span></span></h2><section><ul><li><p><span>No medical SNPs; so avoid SNPs that could be associated with heavy
  medical information</span></p></li><li><p><span>No SNPs that could potentially give you PTSD</span></p></li></ul></section></div><div class="firn-headline-section firn-headline-section-2"><h2 class="firn-headline firn-headline-2" id="the-reaction"><span class="firn-headline-text"><span>the Reaction</span></span></h2><section><ul><li><p><span>Each student's PCR reaction will only amplify 500-800 nucleotide
  region</span></p></li><li><p><span>Find SNPs that have specific traits independently, because most SNPs
  are polygenic (they tell you info only if you discover many different
  SNPs, which is complicated and hard)</span></p></li><li><p><span>SNPs are sometimes discovered using Genome Wide Association Studies,
  which is challenging to deal with because they may be flawed</span></p></li></ul><hr /></section></div><div class="firn-headline-section firn-headline-section-2"><h2 class="firn-headline firn-headline-2" id="polymerase-chain-reaction-(pcr)"><span class="firn-headline-text"><span>Polymerase Chain Reaction (PCR)</span></span></h2><section><p><em><span>A reaction by which the DNA replication process is attempted to be
replicated out of the cell in preparation to sequence one's DNA</span></em></p><p><strong><span>PCR provides a way of creating </span><em><span>lots</span></em><span> of copies of a </span><em><span>small</span></em><span> strand of
DNA</span></strong></p></section><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="reaction-need"><span class="firn-headline-text"><span>Reaction Need</span></span></h3><section><ul><li><p><span>The DNA Portion (duh)</span></p></li><li><p><span>A buffer to host the liquid</span></p></li><li><p><span>Primers - Bit of RNA to help DNA polymerase get going</span></p></li><li><p><span>DNA Polymerase - The Enzyme to Copy the DNA => Typically, the TAQ
  Polymerease is used because it is from a bacterial that's quite
  resistant to heat</span></p></li><li><p><span>DNA Nucleotides - to build the replicated DNA</span></p></li></ul></section></div><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="steps"><span class="firn-headline-text"><span>Steps</span></span></h3><section><ol><li><p><span>Denaturation</span></p><ol><li><p><span>Cook the DNA ("denature it"/heat it) to seperate the two strands
      of the DNA</span></p></li></ol></li><li><p><span>Annealing</span></p><ol><li><p><span>Cool the Cooked DNA, and dump in the primers</span></p></li><li><p><span>The specific temperature in this step is important because it
      controlls where the primer binds, and hence which part of the DNA
      you want to amplify</span></p></li></ol></li><li><p><span>DNA Synthesis => make more copies of the DNA</span></p><ol><li><p><span>This needs a slightly higher temperature than annealing, but which
      one depends on what polymerase you used</span></p></li></ol></li><li><p><span>Go to step 1</span></p></li></ol><p><span>Note: the DNA polymerase may run over, so we just keep repeating this
and it will slowly become more successful as the DNA polymerease repeats
its work repeatedly.</span></p><p><span>BTW: if you read carefully, you will realize that you made TWO DNA
double-strands out of a single double strand. So exponential growth
baybee and then you could get lots of copies of that region.</span></p></section></div><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="but,-why?"><span class="firn-headline-text"><span>But, why?</span></span></h3><section><p><span>In order to have enough copies for gel electroprorisis (like, the
rRT-PCR COVID test) or, for things like crime forensics, you need many
copies of the same DNA for a clearly detectable result for analysis that
may have a high threshold.</span></p></section></div></div></div></div></div></article><div class="metapanel"><div class="metalabel">Contents</div><ol><li><a href="#snp">SNP</a><ol><li><a href="#snp-guidelines">SNP Guidelines</a></li><li><a href="#the-reaction">the Reaction</a></li><li><a href="#polymerase-chain-reaction-(pcr)">Polymerase Chain Reaction (PCR)</a><ol><li><a href="#reaction-need">Reaction Need</a></li><li><a href="#steps">Steps</a></li><li><a href="#but,-why?">But, why?</a></li></ol></li></ol></li></ol><div class="metalabel">Backlinks</div><ul class="firn-backlinks"><li class="firn-backlink"><a href="/biology/bio10/index">Biology Master Index</a></li></ul></div></main></body></html>