<html><head><meta charset="utf-8" /><meta content="width=device-width, initial-scale=1.0" name="viewport" /><meta content="Taproot Authors" name="author" /><meta content="A collection of academic and project notes" name="description" /><title>Taproot</title><link href="/static/css/firn_base.css" rel="stylesheet" /><link href="/static/css/prism.css" rel="stylesheet" /><link href="/static/css/global.css" rel="stylesheet" /><script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_HTML" type="text/javascript"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/prism.min.js" type="text/javascript"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
        'displayAlign': 'center',
        'displayIndent': '0em',
        'extensions': ['tex2jax.js'],
        'tex2jax': {
        'inlineMath': [ ['$','$'], ['\\(','\\)'] ],
        'processEscapes': true
        },
  tex: {
    autoload: {
      cases: [[], ['numcases', 'subnumcases']]
    }
  },
        'HTML-CSS': { scale: 100,
                        linebreaks: { automatic: 'false' },
                        webFont: 'TeX'
                       },
        SVG: {scale: 100,
              linebreaks: { automatic: 'false' },
              font: 'TeX'},
        NativeMML: {scale: 100},
        TeX: { equationNumbers: {autoNumber: 'AMS'},
               MultLineWidth: '85%',
               TagSide: 'right',
               TagIndent: '.8em'
             }})</script></head><body><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/components/prism-core.min.js" type="text/javascript"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/plugins/autoloader/prism-autoloader.min.js" type="text/javascript"></script><div class="headerline"><a class="wordmark" href="https://taproot3.sanity.gq/" style="border:0">TR3</a></div><main><article class="content rss"><div class="preamble"><h1 class="title">SNP Final Project</h1></div><div class="metamble"><span><span>Written by </span><span>Huxley</span></span></div><div class="notebody"><div><section><p><span>#ret #ref</span></p><hr /></section><div class="firn-headline-section firn-headline-section-1"><h1 class="firn-headline firn-headline-1" id="snp-project-write-up"><span class="firn-headline-text"><span>SNP Project Write-up</span></span></h1><section><p><span>%% Resources: </span><a class="firn-internal" href="/biology/bio10/KBxSNPPCR">KBxSNPPCR</a><span></span><a class="firn-external" href="https://docs.google.com/document/d/1SRQSvppoSJYlOfJvIKg3INx79Yzv5MmXcFcNG1-nIwE/edit" target="_blank">Instructions</a></p></section><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="init-planning"><span class="firn-headline-text"><span>Init Planning</span></span></h3><div class="firn-headline-section firn-headline-section-6"><h6 class="firn-headline firn-headline-6" id="outline"><span class="firn-headline-text"><span>Outline</span></span></h6><section><ul><li><p><span>basics</span></p></li><li><p><span>function and regulation</span></p></li><li><p><span>SNP effect</span></p></li></ul></section></div><div class="firn-headline-section firn-headline-section-6"><h6 class="firn-headline firn-headline-6" id="writing!"><span class="firn-headline-text"><span>Writing!</span></span></h6></div></div><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="part-one!-org"><span class="firn-headline-text"><span>Part One! org</span></span></h3><section><p><span>The </span><em><span>COMT</span></em><span> gene, or catechol-O-methyltransferase, encodes the </span><em><span>COMT</span></em><span>
enzyme which is responsible for breaking down neurotransmitters the
brain's prefrontal cortex. More specifically, it acts as a catalyst for
the transfer of a methyl group from S-adenosylmethionine to dopamine,
epinephrine, and norepinephrine. This process, called O-methylation,
leads to the degradation of the aforementioned neurotransmitters. The</span><em><span>COMT</span></em><span> enzyme also effects the metabolism of exogenous substances, but
that is irrelevant for the mutation at hand</span><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/gene/1312" target="_blank">citation</a><span>. The </span><em><span>COMT</span></em><span> gene
itself is 27.22kb long and located on chromosome 22q11.2</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/1572656/" target="_blank">citation</a><span>. It has
ubiquitous expression in 27 tissues, including the placenta, the
adrenal, and the lung</span><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression" target="_blank">citation</a><span>.
Val158Met, also known as rs4680, is a common missense mutation swapping
a guanine for an adenine. It has the frequency G=0.510915, and thus,
A=0.489085. Val158Met causes the </span><em><span>COMT</span></em><span> enzyme to be roughly 25% as
effective compared to the wild type. Expression levels in mRNA, despite
its reduced protein abundance, are not effected by Val158Met</span><a class="firn-external" href="https://www.hindawi.com/journals/dm/2020/8850859/" target="_blank">citation</a><span> &</span><a class="firn-external" href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0" target="_blank">citation</a><span>.
Thus, Val158Met must be located in a protein-coding region causing the</span><em><span>COMT</span></em><span> enzyme to have lower protein integrity, explaining the
discrepancy between mRNA expression and protein expression. This lower
protein integrity is most likely manifested as diminished
thermostability of the enzyme</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/7703232/" target="_blank">citation</a><span>. The higher level
effect of this reduced enzyme efficacy is greatly debated, and linked to
many different phenotypes. At a broad level, lower </span><em><span>COMT</span></em><span> activity leads
to higher levels of catecholamines in the prefrontal cortex. The actual
effect of these increased levels are not well understood. One proposed
theory is the Warrior versus Worrier hypothesis, which outlines two
groups of personality traits based upon the Val158Met mutation</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/17008817/" target="_blank">citation</a><span>. The "Warrior"
group, defined as the wild type group with lower levels of
catecholamines like dopamine, are said to have an advantage in
processing aversive stimuli. They are also said to have higher pain
tolerance, be less prone to stress, less exploratory, and ect. However,
many of these claims are not well defined and bordering on pseudoscience</span><a class="firn-external" href="https://selfdecode.com/snp/rs4680/" target="_blank">citation</a><span>. The "Worrier" group,
those with the mutation, are said to have an advantage in memory and
attention tasks </span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/17008817/" target="_blank">citation</a><span>.
The Val158Met mutation has also been linked to schizophrenia, but this
claim is debated</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/32931693/" target="_blank">citation</a><span>.</span></p><hr /></section><div class="firn-headline-section firn-headline-section-4"><h4 class="firn-headline firn-headline-4" id="citations:"><span class="firn-headline-text"><span>Citations:</span></span></h4><section><p><em><span>(order of appearance)</span></em></p><ul><li><p><span>[[https://www.ncbi.nlm.nih.gov/gene/1312][NCBI COMT
  catechol-O-methyltransferase [ </span><em><span>Homo sapiens</span></em><span> (human) ]]] -
  </span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/1572656/" target="_blank">Chromosomal mapping of the
  human catechol-O-methyltransferase gene to 22q11.1----q11.2</a></p></li><li><p><span>[[https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression][Gene
  Expression -- NCBI COMT catechol-O-methyltransferase [ </span><em><span>Homo sapiens</span></em><span>
  (human) ]]]</span></p></li><li><p><a class="firn-external" href="https://www.hindawi.com/journals/dm/2020/8850859/" target="_blank">Lack of
  Association between rs4680 Polymorphism in
  Catechol-O-Methyltransferase Gene and Alcohol Use Disorder: A
  Meta-Analysis</a></p></li><li><p><a class="firn-external" href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0" target="_blank">Functional
  Analysis of Genetic Variation in Catechol-O-Methyltransferase (COMT):
  Effects on mRNA, Protein, and Enzyme Activity in Postmortem Human
  Brain</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/7703232/" target="_blank">Kinetics of human soluble
  and membrane-bound catechol O-methyltransferase: a revised mechanism
  and description of the thermolabile variant of the enzyme</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/17008817/" target="_blank">Warriors versus worriers:
  the role of COMT gene variants</a></p></li><li><p><a class="firn-external" href="https://selfdecode.com/snp/rs4680/" target="_blank">Self Decode rs4680</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/32931693/" target="_blank">The effect of rs1076560
  (DRD2) and rs4680 (COMT) on tardive dyskinesia and cognition in
  schizophrenia subjects</a></p></li></ul><p><span>%%</span></p></section></div></div><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="part-one!"><span class="firn-headline-text"><span>Part One!</span></span></h3><section><p><span>%% ### Feedback and revisions</span></p><p><span>%% The </span><em><span>COMT</span></em><span> gene, or catechol-O-methyltransferase, encodes the </span><em><span>COMT</span></em><span>
enzyme which is responsible for breaking down neurotransmitters the
brain's prefrontal cortex. More specifically, it acts as a catalyst for
the transfer of a methyl group from S-adenosylmethionine to dopamine,
epinephrine, and norepinephrine. This process, called O-methylation,
leads to the degradation of the aforementioned neurotransmitters. The</span><em><span>COMT</span></em><span> enzyme also effects the metabolism of exogenous substances, but
that is irrelevant for the mutation at hand</span><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/gene/1312" target="_blank">citation</a><span>. The </span><em><span>COMT</span></em><span> gene
itself is 27.22kb long and located on chromosome 22q11.2</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/1572656/" target="_blank">citation</a><span>. It has
ubiquitous expression in 27 tissues, including the placenta, the
adrenal, and the lung</span><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression" target="_blank">citation</a><span>.
This expression is dynamically regulated during brain development and
due to environmental stimuli. Despite much research into COMT
regulation, the actual processes are still mostly unknown</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/21095457/" target="_blank">citation</a><span>. COMT has two
promoters --- P2 functions constitutively, whereas P1 has tissue
dependent regulation. This tissue specific regulation is most likely
done by C/EBPalpha.</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/8672242/" target="_blank">citation</a><span>. Val158Met, also
known as rs4680, is a common missense mutation swapping a guanine for an
adenine. It has the frequency G=0.510915, and thus, A=0.489085.
Val158Met causes the </span><em><span>COMT</span></em><span> enzyme to be roughly 25% as effective
compared to the wild type. Expression levels in mRNA, despite its
reduced protein abundance, are not effected by Val158Met</span><a class="firn-external" href="https://www.hindawi.com/journals/dm/2020/8850859/" target="_blank">citation</a><span> &</span><a class="firn-external" href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0" target="_blank">citation</a><span>.
Thus, Val158Met must be located in a protein-coding region causing the</span><em><span>COMT</span></em><span> enzyme to have lower protein integrity, explaining the
discrepancy between mRNA expression and protein expression. This lower
protein integrity is most likely manifested as diminished
thermostability of the enzyme, in turn leading to its reduced
effectiveness </span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/7703232/" target="_blank">citation</a><span>.
The higher level effect of this reduced enzyme efficacy is greatly
debated, and linked to many different phenotypes. At a broad level,
lower </span><em><span>COMT</span></em><span> activity leads to higher levels of catecholamines in the
prefrontal cortex. The actual effect of these increased levels are not
well understood. One proposed theory is the Warrior versus Worrier
hypothesis, which outlines two groups of personality traits based upon
the Val158Met mutation</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/17008817/" target="_blank">citation</a><span>. The "Warrior"
group, defined as the wild type group with lower levels of
catecholamines like dopamine, are said to have an advantage in
processing aversive stimuli. They are also said to have higher pain
tolerance, be less prone to stress, less exploratory, and ect. However,
many of these claims are not well defined and bordering on pseudoscience</span><a class="firn-external" href="https://selfdecode.com/snp/rs4680/" target="_blank">citation</a><span>. The "Worrier" group,
those with the mutation, are said to have an advantage in memory and
attention tasks </span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/17008817/" target="_blank">citation</a><span>.
The Val158Met mutation has also been linked to schizophrenia, but this
claim is debated</span><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/32931693/" target="_blank">citation</a><span>.</span></p></section><div class="firn-headline-section firn-headline-section-5"><h5 class="firn-headline firn-headline-5" id="citations:"><span class="firn-headline-text"><span>Citations:</span></span></h5><section><p><em><span>(order of appearance)</span></em></p><ul><li><p><span>[[https://www.ncbi.nlm.nih.gov/gene/1312][NCBI COMT
  catechol-O-methyltransferase [ </span><em><span>Homo sapiens</span></em><span> (human) ]]]</span></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/1572656/" target="_blank">Chromosomal mapping of the
  human catechol-O-methyltransferase gene to 22q11.1----q11.2</a></p></li><li><p><span>[[https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression][Gene
  Expression -- NCBI COMT catechol-O-methyltransferase [ </span><em><span>Homo sapiens</span></em><span>
  (human) ]]]</span></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/21095457/" target="_blank">The
  Catechol-/O/-Methyltransferase Gene: Its Regulation and
  Polymorphisms</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/8672242/" target="_blank">Characterization of the
  rat catechol-O-methyltransferase gene proximal promoter:
  identification of a nuclear protein-DNA interaction that contributes
  to the tissue-specific regulation</a></p></li><li><p><a class="firn-external" href="https://www.hindawi.com/journals/dm/2020/8850859/" target="_blank">Lack of
  Association between rs4680 Polymorphism in
  Catechol-O-Methyltransferase Gene and Alcohol Use Disorder: A
  Meta-Analysis</a></p></li><li><p><a class="firn-external" href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0" target="_blank">Functional
  Analysis of Genetic Variation in Catechol-O-Methyltransferase (COMT):
  Effects on mRNA, Protein, and Enzyme Activity in Postmortem Human
  Brain</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/7703232/" target="_blank">Kinetics of human soluble
  and membrane-bound catechol O-methyltransferase: a revised mechanism
  and description of the thermolabile variant of the enzyme</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/17008817/" target="_blank">Warriors versus worriers:
  the role of COMT gene variants</a></p></li><li><p><a class="firn-external" href="https://selfdecode.com/snp/rs4680/" target="_blank">Self Decode rs4680</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/32931693/" target="_blank">The effect of rs1076560
  (DRD2) and rs4680 (COMT) on tardive dyskinesia and cognition in
  schizophrenia subjects</a></p></li></ul><hr /></section></div></div><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="part-two"><span class="firn-headline-text"><span>Part Two</span></span></h3><section><p><span>%% </span><em><span>infographic time.</span></em></p><ul><li><p><span>*Create an infographic that diagrams the various connections between
  the gene/SNP genotypic variants and the known phenotypic associations.
  This graphic should visually show the biological effects of the
  gene/protein and studied genotypes on human traits/phenotypes. It is
  also important to highlight the ways in which the environment affects
  gene expression, protein function and/or phenotype (see the example
  infographic for APOE below, which makes connections to diet and
  traumatic brain injury). You can choose to visually organize and/or
  lay out your graphic in a variety of formats, but make sure that the
  following information is included:*</span></p></li><li><p><span>info to include</span></p><ul><li><p><strong><span>gene basics</span></strong></p><ul><li><p><span>Gene info (gene size, location of SNP, SNP variants, SNP
      frequency)</span></p></li><li><p><span>Protein info (protein size, protein variants if known)</span></p></li><li><p><span>reg</span></p><ul><li><p><span>estrogen?</span></p></li><li><p><span>brain develpoment</span></p></li><li><p><span>eviromental</span></p></li><li><p><span>P2:</span></p><ul><li><p><span>constitutively</span></p></li></ul></li><li><p><span>P1:</span></p><ul><li><p><span>tissue dependent, C/EBPalpha, ect.</span></p></li></ul></li></ul></li></ul></li><li><p><strong><span>SNP variants to human phenotype relationships</span></strong></p><ul><li><p><span>reduced COMT enzyme activity</span></p></li><li><p><span>warrior worrier stuff</span></p></li></ul></li><li><p><strong><span>Gene-environment Interactions</span></strong></p><ul><li><p><span>above -> situations that better suite?</span></p></li><li><p><span>gene / enviroment stuff?
      https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3447184/</span></p></li></ul></li></ul></li></ul><p><span>%% Submitted in a separate file. You can also check it out linked</span><a class="firn-external" href="https://www.figma.com/file/CADCiIWAFlqyFZu0Qayj62/COMT?node-id=0%3A1" target="_blank">here</a><span>.
%%</span><a class="firn-internal" href="/biology/bio10/Letter - 1 (1).jpg">Letter - 1 (1).jpg</a><span>%%
%%</span><a class="firn-internal" href="/biology/bio10/COMTVal158Metpt2it1.pdf">COMTVal158Metpt2it1.pdf</a><span>%%</span></p><hr /></section></div><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="part-three"><span class="firn-headline-text"><span>Part Three</span></span></h3><section><p><span>%% #### outlinin: prompt: In this section, you should try to provide
some evolutionary insight on the SNP alleles that we see for your gene
in the human population. You may focus on one particular allele if you
see that it is better-researched. Using actual research into the
evolution of the allele(s) and/or research about gene and gene variant
functions, explain why/how the SNP allele(s) you studied were maintained
in the human population (in over 1% of people studied). At some point
the allele first appeared as a mutation but it subsequently spread into
a relatively large proportion of people; why might that have been? -
Although this section should be based on gene/SNP research and your
understanding of evolution, your explanations may be somewhat
speculative due to the difficulty in obtaining evidence for certain
evolutionary predictions/hypotheses (because environmental pressures,
migrations, and random events that influenced evolution likely happened
long ago). That is okay; just show us that you're using research-backed
reasoning about your allele(s) and that you have an understanding of
evolutionary mechanisms. - In terms of evolutionary mechanisms, you
should be thinking about possible selective pressures that may have
maintained certain alleles in the human population (and disfavored
others). Note that evolution typically operates over long timescales, so
the selective pressures that are most likely to have played a role were
operating before civilizations, agriculture, etc (with some exceptions
that may have evolved more "recently", like lactase persistence,
high-altitude adaptation, and disease resistances). - It's also
important to remember the possible contributions of mechanisms like gene
flow and genetic drift in getting to the allele frequencies that we see
today. It would be harder to detect whether/when these happened without
complex analyses of sequences, but you can still acknowledge their
possible impact and explain how these mechanisms operate.</span></p><p><span>allele: g -> a</span></p><p><span>selection factors:</span></p><p><span>population makes it's own selection factors need for warrior / worrier</span></p><p><span>organism, gene, population</span></p><p><span>greedy epsilon, worrier as epsilon</span></p><p><span>evolution on multiple levels like genetic, epigenetic, symbolic,
cultural, ect</span></p><p><span>fitness landscapes of organisms but also of tech</span></p><p><span>worriers are better for jumping out of local minima warriors are better
for carrying out the best strategy</span></p><p><span>perhaps a collection of personality-ish traits that are all being
balanced</span></p><p><span>research:</span></p><blockquote><p><span>  A comparison of human and mouse COMT confirms that the amino acid at
  the Val/Met locus is important for COMT activity and suggests that
  COMT activity has decreased during the course of evolution.
  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182110/</span></p></blockquote><p><span>derived allele uniqe to humans</span></p><p><span>https://www.jstage.jst.go.jp/article/ase/121/3/121_130731/_html/-char/en</span></p><p><span>%%</span></p><p><span>Not much is known about the evolution of the COMT*L allele. In the sea
of speculation, only two facts emerge: COMT activity has decreased
during the course of recent evolution, and COMT*L is a derived allele
unique to humans</span><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182110/" target="_blank">citation</a><span> &</span><a class="firn-external" href="https://www.jstage.jst.go.jp/article/ase/121/3/121_130731/_html/-char/en" target="_blank">citation</a><span>.
For the following speculation, the phenotypes said to arise from
different variations of the COMT gene will be assumed true. Speculating
about evolution on the organism level is relatively straightforward:
this organism evolved a patch of photosensitive skin so it could tell
which way was up. However, this level of analysis breaks down when
trying to explain altruism, and thus comes speculation on the gene
level: these organisms evolved so they would "lay down [their] life for
two brothers or eight cousins" - </span><em><span>J.B.S. Haldane</span></em><span>. The evolution of
COMT*L is not explained solely by either of these levels --- instead,
its evolution operates on the population level.</span></p><p><span>It can be assumed that given a constant environment without changing
selection factors, there is an optimal mix of so called Warriors and
Worriers. On top of the hypothetically constant selection pressures from
the environment, the population itself creates its own selection
pressures based upon the ratio of Warriors to Worriers. If there are too
many Warriors, then a selection pressure favoring Worriers would arise,
and vice versa, leading the population towards the optimal ratio of
Warriors and Worriers. With heavy speculation, one could suggest that
there exists a whole set of personality traits that are constantly being
balanced across the population. Of course, these selection pressures
would on top of the environmental selection pressures; a Worrier is more
likely to do better in a famine where new ways to get resources need to
created. A Worrier would be more likely to advocate for saving some
grain for the future despite the fact that people are hungry now. A
Warrior would do better in situations like getting chased by a predator,
where they need to act well under stress and have high resilience.</span></p><p><span>Of course, environmental selection pressures are not constant. Hence,
the question becomes, what type of environment shifts the optimal ratio
of Warriors to Worriers? To answer this question, one must think of
evolution in terms of a fitness landscape. An organism can be stuck in a
local minima and require some large mutation to jump out of it.
Disregarding genetic drift, the only other mutations that would persist
would be ones leading the species to the bottom of the local minima. In
recent years, the theory of evolution has been extended into axes beyond
just the organism itself. A good book on the topic is</span><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1265888/" target="_blank">Evolution in
four dimensions: /Genetic, epigenetic, behavioral, and symbolic
variation in the history of life/</a><span>. Each of these axes have their own
fitness landscape associated with them. We can imagine an axis called,
for lack of a better term, technological. This axis could include, of
course, technology, but also human action and innovation. For example,
traveling to this location through this new path, or hunting in this
different patch of land instead of the old patch would be grouped into
this axis.</span></p><p><span>In the fitness landscape associated with this new technological axis,
which co-evolves with the biological evolution of humans, Warriors would
be better at carrying out the current best strategy, whereas the
Worriers would be best at jumping out of the local minima and finding
new strategies. This setup of Warriors and Worriers is much like
Epsilon-Greedy algorithms in the world of computer science. A Greedy
algorithm is an algorithm which simply does the best action for itself
in the current situation, disregarding the future. These algorithms
sometimes work for very complex problems. An Epsilon-Greedy algorithm is
often used when dealing with unknown probability distributions, like
trying to navigate an unknown fitness landscape. It acts just like a
normal Greedy algorithm, except for some epsilon amount of time where it
explores instead of carrying out the current best action. Figuring out
the optimal value of this epsilon is a massive problem in computer
science. In this case, Warriors would be the ones best suited for
carrying out the normal Greedy function, and the Worriers would be the
epsilon. Thus, when the population is still catching up to the limits of
the current strategy --- when times are good --- Warriors will do
better. When a new strategy is needed, Worriers will be needed.</span></p><p><span>However, the fact that COMT activity has been decreasing over time is
still not explained. I would propose that this decrease is due to the
expansion of the adjacent possible. If one imagines the realm of all
that is currently possible as a circle, a ring outside of that circle is
the adjacent possible --- that which is almost achievable. As the
possible grows, the adjacent possible grows faster. As the possible
expands, the adjacent possible becomes much larger. As the adjacent
possible becomes larger, there becomes more local minima to jump out of,
and thus, the optimal ratio of Warriors and Worriers shifts to more
Worriers. Of course, this is all speculation, but until the circle of
possible grows larger, that is all we can do.</span></p></section><div class="firn-headline-section firn-headline-section-5"><h5 class="firn-headline firn-headline-5" id="citations"><span class="firn-headline-text"><span>Citations</span></span></h5><section><ul><li><p><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182110/" target="_blank">Functional
  Analysis of Genetic Variation in Catechol-O-Methyltransferase
  (/COMT/): Effects on mRNA, Protein, and Enzyme Activity in Postmortem
  Human Brain</a></p></li><li><p><a class="firn-external" href="https://www.jstage.jst.go.jp/article/ase/121/3/121_130731/_html/-char/en" target="_blank">Correlation
  of the COMT Val158Met polymorphism with latitude and a hunter-gather
  lifestyle suggests culture--gene coevolution and selective pressure on
  cognition genes due to climate</a></p></li><li><p><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1265888/" target="_blank">Evolution in
  four dimensions: /Genetic, epigenetic, behavioral, and symbolic
  variation in the history of life/</a></p></li></ul><hr /></section></div></div><div class="firn-headline-section firn-headline-section-3"><h3 class="firn-headline firn-headline-3" id="part-four:-combined-citations"><span class="firn-headline-text"><span>Part Four: Combined Citations</span></span></h3><section><p><em><span>(order of appearance)</span></em></p><ul><li><p><span>[[https://www.ncbi.nlm.nih.gov/gene/1312][NCBI COMT
  catechol-O-methyltransferase [ </span><em><span>Homo sapiens</span></em><span> (human) ]]]</span></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/1572656/" target="_blank">Chromosomal mapping of the
  human catechol-O-methyltransferase gene to 22q11.1----q11.2</a></p></li><li><p><span>[[https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression][Gene
  Expression -- NCBI COMT catechol-O-methyltransferase [ </span><em><span>Homo sapiens</span></em><span>
  (human) ]]]</span></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/21095457/" target="_blank">The
  Catechol-/O/-Methyltransferase Gene: Its Regulation and
  Polymorphisms</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/8672242/" target="_blank">Characterization of the
  rat catechol-O-methyltransferase gene proximal promoter:
  identification of a nuclear protein-DNA interaction that contributes
  to the tissue-specific regulation</a></p></li><li><p><a class="firn-external" href="https://www.hindawi.com/journals/dm/2020/8850859/" target="_blank">Lack of
  Association between rs4680 Polymorphism in
  Catechol-O-Methyltransferase Gene and Alcohol Use Disorder: A
  Meta-Analysis</a></p></li><li><p><a class="firn-external" href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0" target="_blank">Functional
  Analysis of Genetic Variation in Catechol-O-Methyltransferase (COMT):
  Effects on mRNA, Protein, and Enzyme Activity in Postmortem Human
  Brain</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/7703232/" target="_blank">Kinetics of human soluble
  and membrane-bound catechol O-methyltransferase: a revised mechanism
  and description of the thermolabile variant of the enzyme</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/17008817/" target="_blank">Warriors versus worriers:
  the role of COMT gene variants</a></p></li><li><p><a class="firn-external" href="https://selfdecode.com/snp/rs4680/" target="_blank">Self Decode rs4680</a></p></li><li><p><a class="firn-external" href="https://pubmed.ncbi.nlm.nih.gov/32931693/" target="_blank">The effect of rs1076560
  (DRD2) and rs4680 (COMT) on tardive dyskinesia and cognition in
  schizophrenia subjects</a></p></li><li><p><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182110/" target="_blank">Functional
  Analysis of Genetic Variation in Catechol-O-Methyltransferase
  (/COMT/): Effects on mRNA, Protein, and Enzyme Activity in Postmortem
  Human Brain</a></p></li><li><p><a class="firn-external" href="https://www.jstage.jst.go.jp/article/ase/121/3/121_130731/_html/-char/en" target="_blank">Correlation
  of the COMT Val158Met polymorphism with latitude and a hunter-gather
  lifestyle suggests culture--gene coevolution and selective pressure on
  cognition genes due to climate</a></p></li><li><p><a class="firn-external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1265888/" target="_blank">Evolution in
  four dimensions: /Genetic, epigenetic, behavioral, and symbolic
  variation in the history of life/</a></p></li></ul><p><span>%% ### More Feedback</span></p><blockquote><p><span>  Huxley, Sorry, I should have been a bit more clear. I was asking you
  to revise the second paragraph of Part 3 to fully explain your
  reasoning there. If that's what you were trying to do in this comment,
  then yes I do think you can clarify further before incorporating it
  into the write-up: 1) Are you still using epsilon in the sense of the
  epsilon-greedy algorithm that you described, essentially "exploratory"
  behavior (in contrast to "exploitative" behavior)? If you're going to
  introduce this epsilon concept earlier in this Part (i.e. the second
  paragraph), you'd obviously need to define your terms. Or you could
  probably get your point across here without the analogy to the
  algorithm. 2) It sounds like you might be proposing that there is
  group selection at work in maintaining both alleles in the population.
  It also sounds a bit like you're invoking the idea of balancing
  selection. If you are indeed thinking along these lines, can you
  ground your explanations in relation to those concepts? If you're not
  familiar with those and this is a separate line of speculation, then
  the areas in your reasoning that could use more explanation are: -why
  it can be assumed that there is some optimal mix of "warriors" and
  "worriers." How does that relate to both individual and group fitness
  and how do you expect those to interact? -how the balancing mechanism
  that you're proposing at the population level would actually work,
  i.e. what is the selection pressure that would tip the scales toward
  one phenotype and then back to the other based on changing ratios.</span></p></blockquote><p><span>%%</span></p></section></div></div></div></div></article><div class="metapanel"><div class="metalabel">Contents</div><ol><li><a href="#snp-project-write-up">SNP Project Write-up</a><ol><li><a href="#init-planning">Init Planning</a><ol><li><a href="#outline">Outline</a></li><li><a href="#writing!">Writing!</a></li></ol></li></ol></li></ol><div class="metalabel">Backlinks</div><ul class="firn-backlinks"><li class="firn-backlink"><a href="/biology/bio10/index">Biology Master Index</a></li></ul></div></main></body></html>